\chapter{Endoparasitoid lifestyle promotes endogenization and domestication of dsDNA viruses}
    \label{chap:Domestication_within_parasitoids}

    \begin{center}
        \Large Benjamin guinet$^{\text{1}}$, David Lepetit$^{\text{1}}$, David G Notton$^{\text{7}}$, Elijah J Talamas$^{\text{8}}$, Jean-Yves Rasplus$^{\text{2}}$, Julia Stigenberg$^{\text{5}}$ , Lene Sigsgaard$^{\text{3}}$, Nicolas Ris$^{\text{4}}$, Peter N$^{\text{6}}$, Sylvain Charlat$^{\text{1}}$, Damien de Vienne$^{\text{1}}$, Bastien Boussau$^{\text{1}}$, Julien Varaldi$^{\text{1}}$\\
        \vspace{0.5cm}
        \scriptsize
        $^{\text{1}}${Université Lyon 1, CNRS, Laboratoire de Biométrie et Biologie Evolutive UMR 5558, F-69622 Villeurbanne, France.}\\
        $^{\text{2}}${INRA, UMR1062 CBGP, F-34988 Montferrier-sur-Lez, France.}\\
        $^{\text{3}}${Department of Plant and Environmental Sciences, University of Copenhagen, Thorvaldsensvej 40, 1871 Frederiksberg C, Denmark.}\\
        $^{\text{4}}${INRAE, Institut Sophia-Agrobiotech, Université Côte d'Azur, CNRS, 06903 Sophia Antipolis, France.}\\
        $^{\text{5}}${Department of Zoology, Swedish Museum of Natural History, Box 50007, 104 05 Stockholm, Sweden.}\\
        $^{\text{6}}${Zoological Museum, Department of Entomology, University of Copenhagen,
        Universitet sparken 15, DK-2100 Copenhagen, Denmark.}\\
        $^{\text{7}}${Natural Sciences Department, National Museums Collection Centre, 242 West Granton Road, Granton, Edinburgh, EH5 1JA, United Kingdom.}\\
        $^{\text{8}}${Department of Plant Industry, Florida Department of Agriculture and Consumer Services, Gainesville, FL 32399, USA.}\\
    \end{center}
     \setcounter{minitocdepth}{1}

    {\hypersetup{linkcolor=GREYDARK}\minitoc}
\clearpage

\label{sec:chap1}

\section{Reproducibility and Supplemental material}
All additional scripts and large figures are available at the following GitHub link : \newline (\href{https://github.com/BenjaminGuinet/PhD_defense/tree/main/Supplementary_paper1}{Supplementary\_paper1}).

\section{Abstract}

The accidental endogenization of viral elements within eukaryotic genomes can occasionally provide significant evolutionary benefits, giving rise to their long-term retention, that is, to viral domestication. For instance, in some endoparasitoid wasps (whose immature stages develop inside their hosts), the membrane-fusion property of double-stranded DNA viruses has been repeatedly domesticated following ancestral
endogenizations. The endogenized genes provide female wasps with a delivery tool to inject virulence factors that are essential to the developmental success of their offspring. Because all known cases of viral domestication involve endoparasitic wasps, we hypothesized that this lifestyle, relying on a close interaction between individuals, may have promoted the endogenization and domestication of viruses. By analyzing the composition of 124 Hymenoptera genomes, spread over the diversity of this clade and including free-living, ecto- and endoparasitoid species, we tested this hypothesis. Our analysis first revealed that double-stranded DNA viruses, in comparisons with other viral genomic structures (ssDNA, dsRNA, ssRNA), are more often endogenized and domesticated (that is, retained by selection) than expected from their estimated abundance in insect viral communities. Secondly, our analysis indicates that the rate at which dsDNA viruses are endogenized is higher in endoparasitoids than in ectoparasitoids or free-living hymenopterans, which also translates into more frequent events of domestication. Hence, these results are consistent with the hypothesis that the endoparasitoid lifestyle has facilitated the endogenization of dsDNA viruses, in turn increasing the opportunities of domestications that now play a central role in the biology of many endoparasitoid lineages.


\section{Introduction}

The recent boom of genome sequencing programs has revealed the abundance of DNA fragments of viral origin within eukaryotic genomes. These so-called Endogenous Viral Elements (EVEs) stem from endogenization events that not only involve retroviruses as donors (as could be expected from their natural lifecycle) but also viruses that do not typically integrate into their host chromosomes \citep{katzourakis_endogenous_2010,feschotte_endogenous_2012,aswad_evolutionary_2021}. In insects, where retroviruses have yet to be found, endogenization events have involved various non-retroviral viruses: three families of large double-stranded (ds) DNA viruses, at least 22 families of RNA viruses, and three families of single-stranded (ss) DNA viruses \citep{gilbert_diversity_2022}. Degeneracy and loss is likely the fate of most EVEs, since they do not \textit{a priori} benefit their hosts. Still, several studies have reported that EVEs can be retained by selection, thus becoming \textit{domesticated} \citep{koonin_depths_2018}. The functions involved include defensive properties against related viruses in mosquitoes \citep{yan_origin_2009,suzuki_non-retroviral_2020}, against macroparasites in some Lepidoptera \citep{gasmi_horizontally_2021}, or modifications in the expression of genes involved in dispersal in aphids \citep{parker_laterally_2019}. Beyond insects, the membrane fusion capacity of viruses, allowing their entry into host cells, has been repeatedly co-opted in three metazoan clades: mammals, viviparous lizards and parasitoid wasps. In placental mammals and viviparous Scincidae lizards, domestication of the \textit{syncytin} protein from retroviruses has allowed the emergence of the placenta, through the development of the syncytium (composed of fused cells) involved in metabolic exchanges between the mother and the fetus \citep{lavialle_paleovirology_2013,cornelis_endogenous_2017}. A similar fusogenic property was repeatedly co-opted by parasitoids belonging to the Hymenoptera order through the  endogenization and domestication of complex viral machineries deriving from large dsDNA viruses  \citep{drezen_endogenous_2017, gilbert_diversity_2022}. The numerous retained viral genes allow parasitoid wasps to produce virus-like structures (VLS) within their reproductive apparatus. These are injected into the wasp's host, together with their eggs, and protect the wasp progeny against the host immune response. This protection is achieved thanks to the ability of VLS to deliver virulence factors in the form of genes (in which case VLS are called polydnavirus - PDV) or proteins (in which case VLS are called Virus-like particles - VLPs) to host immune cells (reviewed in \citep{gauthier_recurrent_2018, drezen_bracoviruses_2021}). So far, 5 independent cases of such viral domestication have been detected in parasitoid wasps, four of them falling within the Ichneumonoidea superfamily \citep{bezier_polydnaviruses_2009,volkoff_analysis_2010,pichon_recurrent_2015,burke_common_2019} and one in the Cynipoidea superfamily (Di Giovanni et al., 2020). The four cases where the donor virus family has been unequivocally identified point towards dsDNA viruses. More specifically, the domesticated EVEs (hereafter, dEVEs) derive from the \textit{Nudiviridae} family in three cases \citep{bezier_polydnaviruses_2009,pichon_recurrent_2015,burke_common_2019} while the forth involves a putatively new viral family denoted "LbFV-like" (Di Giovanni et al., 2020). Notably, all these domestication events took place in endoparasitoids, that is, in species that deposit their eggs inside the hosts, as opposed to ectoparasitoids that lay on their surface.

Beyond these well characterized events of viral domestication in Hymenoptera, additional cases of endogenization have been uncovered, in studies that enlarged the taxonomic focus of either the hosts \citep{ter_horst_endogenous_2019, cheng_nudivirus_2020,kondo_novel_2019} or the viruses that were considered  \citep{flynn_assessing_2019,ter_horst_endogenous_2019,irwin_systematic_2022,li_hgt_2022}. Here, we complement this earlier work by expanding the range of both the hosts and viruses under study, and by further analyzing which endogenization cases have been followed by a domestication event.

To this end, we developed a bioinformatic pipeline to detect endogenization events involving any kind of viruses (DNA/RNA, single-stranded, double-stranded), at the scale of the whole Hymenoptera order. This analysis first allowed us to test whether the propensity of viruses to enter Hymenoptera genomes, and to be domesticated, depend on their genomic structure (in line with the pattern observed so far, where only dsDNA viruses have been involved in domestication events as described above). 
We then tested whether the lifestyle of the species (free-living, endoparasitoid, ectoparasitoid) correlates with their propensity to integrate and domesticate viruses. Our working hypothesis was that the endoparasitoid lifestyle may be associated with a higher rate of viral endogenization and / or a higher rate of domestication events, for two non-exclusive reasons related either to the exposure  to new viruses and the adaptive value of the endogenized elements.

First, a higher endogenization rate may simply stem from a higher exposure to viruses. Such an effect could be at play in endoparasitoids due to the intimate interaction between the parasitoid egg or larva and the host.  In other words, the endo-parasitic way of life may facilitate the acquisition of new viruses deriving from the hosts. Notably, this lifestyle may also facilitate the maintenance and spread of newly acquired viruses within wasp populations. Indeed, endoparasitoid wasps often inject not only eggs but also venomic compounds (typically produced in the venom gland or in calyx cells) where viruses can be present and may thus be vertically transmitted \citep{martinez_additional_2016}).  In addition, the confinement of the several  developing wasps within a single host may facilitate viral horizontal transmission  and its subsequent spread in wasps populations (e.g. \citep{varaldi_infectious_2003}).

Second, a higher rate of domestication in endoparasitoids may be the consequence of a particular selective regime. This is expected since, these insects are facing the very special challenge of resisting the host immune system, contrary to other lifestyles. This selective pressure may promote the co-option of viral functions such as the above-mentioned membrane fusion activity, that provide a very effective mean to deliver virulence factors. 

Our analysis reveals numerous new instances of endogenization events, some of which are also characterized by signatures of molecular domestication. We found a clear enrichment in endogenization events deriving from dsDNA viruses as compared to those with other genomic structures. While the data did not reveal a significant effect of Hymenoptera lifestyles on the acquisition of dsRNA, ssRNA or ssDNA viruses, it supports the hypothesis that genes from dsDNA viruses are more often endogenized and domesticated in endoparasitoids than in free-living and ectoparasitoid species.

%\end{multicols}

\afterpage{%
\onecolumn
\begin{figure}[!htpb]
\captionsetup{font=footnotesize}
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Main_phylogeny_figure2.pdf}
\caption[Paper1:EVE distribution along the Hymenoptera phylogeny]{\textbf{Endogenous Viral Elements and their domestication status in Hymenoptera. }
Lifestyles are displayed next to species names (blue: free-living, green: endoparasitoid, yellow: ectoparasitoid, grey: unknown). The number of EVEs and domesticated EVEs (dEVEs) found in each species are represented respectively by the first and second facet of the horizontal histograms. Colors along these histograms indicate the potential donor viral families (where blue tones correspond to viral dsDNA viruses, red tones to ssDNA viruses, orange/brown tones to dsRNA viruses and green tones to ssRNA genomes). Endogenous Viral elements (EVEs) shared by multiple species and classified within the same event are represented by circles whose size is proportional to their number; those that are considered as domesticated (dEVEs) are surrounded by a black border. Numbers in the white boxes correspond to the number of endogenization events inferred. As an example, \textit{Megastigmus dorsalis} and \textit{Megastigmus stigmatizans} are ectoparasitoids (yellow) sharing a common endogenization event (within the Cluster21304) that likely originated from an unclassified dsRNA virus (grey color in circle), and shows no sign of domestication (no black border around the grey part of the circle). The figure was inspired from the work of \citep{peters_evolutionary_2017}. Details on the phylogenetic inference and time calibration can be found in the Materiel and Methods section; bootstrap information can be found in \tablename{\ref{tab:Summary_candidate_numbers}}; details on lifestyle assignation can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/Assembly_genome_informations.csv}{Assembly\_genome\_informations.csv}.}
\label{figure:Event_phylogeny}
\end{figure}

\clearpage
}

\section{Results}

We screened for EVEs 124 Hymenoptera genome assemblies, including 24 ectoparasitoids, 37 endoparasitoids and 63 free-living species (the list can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/Assembly_genome_informations.csv}{Assembly\_genome\_informations.csv}). EVEs were identified using a sequence-homology approach based on a comprehensive viral protein database. Different confidence levels (ranging from A to D) were associated with the various EVEs inferred, where the A score indicates a maximal confidence level for endogenization. This confidence index is based upon sequencing depth combined with the presence of eukaryotic genes and/or transposable elements in the genomic environment of the candidate loci (as detailed in the Material and Methods section). By default, the four categories are included in the analysis, but unless otherwise stated statistical tests based on the A category only led to the same conclusions (see \figurename{\ref{figure:Bioinformatic_pipeline}}-7 for more details). Our analysis further included an inference of the phylogenetic relationships among homologous EVEs, that was used to map endogenization events on the Hymenoptera species tree. Finally, inferences of domestication events relied upon signatures of purifying selection in the integrated genes (based on \textit{dN/dS} estimates) and/or on expression data.

An important objective of our analysis is to detect and enumerate not only endogenous viral elements (EVEs) but also endogenization \textit{events} that can explain the presence of these EVEs. Indeed, an EVE denotes a single gene of viral origin in a single species. Several neighboring EVEs in a genome most likely result from the endogenization of a single viral genome, and  homologous EVEs shared by several closely related species may further stem from a single ancestral endogenization events. This distinction is critical when it comes to examining the effect of various factors on the probability of integrating EVEs, which implies counting events rather than EVEs. As an example, consider the \textit{Leptopilina} case  (Di Giovanni et al., 2020), involving 13 EVEs shared by 3 closely related species. In this wasp genus, based on previous findings, we expect the 39 EVEs to be grouped into a single endogenization event. Our pipeline appropriately detected 36 EVEs (out of 39) and correctly aggregated them into a single endogenization event mapped on the branch leading to the \textit{Leptopilina} genus. Furthermore, because some of the genes involved are inferred as domesticated, this event is appropriately classified as a domestication event (see \figurename{ \ref{figure:Event_phylogeny}} and \figurename{ \ref{figure:Event_assignation_examples}} for more canonical examples).  
In total, the pipeline correctly detected 88.4\% (152/172) of the EVEs involved in our four "positive controls", previously described as mediating the protection of young wasps  against their host immune system. Among them, 71.82\% were inferred as being domesticated. Out of the 152 positive controls EVEs, 147  were grouped into 4 independent endogenization events, as was expected. The remaining 5 genes had peculiar histories that led our pipeline to infer two additional spurious events (\tablename{\ref{tab:Control_table}}). All detailed results regarding EVEs and dEVEs can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/All_EVEs_dEVEs_informations.txt}{All\_EVEs\_dEVEs\_informations.txt}.

\subsection{Endogenizations involve all viral genomic structures}

A total of 1261 EVEs have been inferred in the whole dataset (\tablename{\ref{tab:Summary_candidate_numbers}},\figurename{\ref{figure:Event_phylogeny}}). These come from 367 endogenization events, the majority of which involved ssRNA and dsDNA viruses (41\% and 35\%, respectively) (\tablename{\ref{tab:Summary_candidate_numbers}}). Among the tip branches leading to the 124 species under study, 113 underwent one endogenization event, with a maximum of 14 events and a median of 3 (\figurename{\ref{figure:Event_phylogeny}}). In total, 91\% of the events (331) occurred on tip branches, and the remaining 9\% are shared by at least two closely related species (\tablename{\ref{tab:Summary_candidate_numbers}}, \figurename{\ref{figure:Event_phylogeny}}). To assess the validity of the procedure used to aggregate multiple EVEs into a single shared ancestral endogenization, we assessed whether EVEs inferred as homologous shared a common genomic environment. We thus tested for the presence of homologous loci in descendant species around the shared ancestral EVEs (using blastn searches between the corresponding scaffolds, see details in \hyperref[sec:MM-6]{Materials and methods}). Among the 36 endogenization events that involved at least two species, 31 were found to carry more homologous loci around the insertion sites than expected by chance (see details in \hyperref[sec:MM-6]{Materials and methods}). Notably, the majority of endogenization events involved a single EVE (a single gene) and only 12 (all from dsDNA viruses), involved the concomitant integration of more than 4 viral genes (\figurename{\ref{figure:Genomic_structure_and_families_Events_distributions}}-C).  

A total of 40 different viral clades (usually families) were inferred as putative donors. Most of them (34) are known to infect insects (\figurename{\ref{figure:Genomic_structure_and_families_Events_distributions}}-B) and these account for the majority of the 331 endogenization events.  However, we found 36 EVEs (24 endogenization events), including 20 high-confidence ones (A-ranked), that derived from 6 viral families not previously reported to infect insects (\textit{Phycodnaviridae}, \textit{Herpesviridae}, \textit{Caulimoviridae}, \textit{Asfaviridae}, \textit{Bornaviridae} and \textit{Mypoviridae}). However, in those cases, the true viral donors may belong to unknown clades that do infect insects. Indeed, although the homology with viral proteins was convincing (median e-value was 9.4095e-12 [min = 9.212e-129, max = 3.305e-08]), the average percentage identity was relatively low (38\% [min = 23.2\%, max = 79.1\%], suggesting that these loci may originate from unknown viruses that are only distantly related to their closest relatives in public databases.


\begin{figure}[!htbp]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Genomic_structure_and_families_Events_distributions.pdf}
\caption[Paper1:Distribution of Events and dEvents among viral genomic structures and viral families]{\textbf{Endogenization involves all types of viral genomic structures.} In all three panels, a yellow color indicates endogenization events that have not been followed by domestication, while orange indicates domestication events. \textbf{A} : Distribution of the number of events inferred, according to the four categories of viral genomic structures. The crosses refer to the expected number of endogenization events for each category based on its estimated relative abundance in insects (see details in \hyperref[sec:MM-8]{Materials and methods} and virus-infecting data in Excel tab:All\_virus\_infecting\_insects\_informations).  \textbf{B} :  Distribution of the various viral families involved in endogenization events. The polarity of ssRNA viruses  is displayed next to the family name. Events involving multiple putative families (i.e. where several viral families are present in the same scaffold) have been excluded from the count. The star next to the family name indicates that the viral family is known to infect insects. \textbf{\text{C}} : Distribution of the number of EVEs per event across viral categories.}
\label{figure:Genomic_structure_and_families_Events_distributions}
\end{figure}

\subsection{Double-stranded DNA viruses are over-represented in endogenization events}

Most of the endogenization events recorded involve ssRNA and dsDNA viruses. But do these proportions simply mirror the diversity and respective abundances of the different kinds of viruses encountered by insects?  The analysis summarized in (\figurename{\ref{figure:Genomic_structure_and_families_Events_distributions}}-A) (see details in \hyperref[sec:MM-8]{Materials and methods}) indicates this is not the case. More specifically, it shows that dsDNA viruses are more frequently endogenized than expected on the basis of their representation in the databases, while ssRNA viruses are under-represented  ($\chi^2$ = 213.36 and 221.38, respectively, for endogenization events and domestication events, d.f. = 3, both p-value $<$ 2.2e-16). Notably, this result is not purely driven by the presence in our data set of the four positive controls (previously described cases of viral domestication, that all involve dsDNA viruses as donors). Finally, among endogenization events involving ssRNA viruses, we found an over-representation of negative stranded ssRNA compared to their relative abundance in public databases (72.2\% compared to 32.6\% in the databases, $\chi^2$=145.87, d.f.=1, p-value $<$ 2.2e-16; see \hyperref[sec:SI-1]{supplemental information} for a discussion).\\

\begin{figure}[!htbp]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Lifecycle_Events_per_genomic_structures.pdf}
\caption[Paper1:Distribution of Events and dEvents among viral genomic structures and between lifestyles]{\textbf{Endogenization and domestication of dsDNA viruses are most prevalent in endoparasitoid species}. \textbf{A}: Distribution of viral endogenization events (Event) and  \textbf{B} of domestication events (dEVEs) across Hymenoptera lifestyles.
Crosses indicate the expected proportion of events associated with the different lifestyles, based on the respective frequencies in our database (ectoparasitoid = 24/124, endoparasitoid = 37/124, free-living = 63/124). The p-values are the results of Fisher's tests comparing the observed and expected distributions. Numbers inside the bars indicate the absolute numbers of events inferred. The ancestral states of the nodes, in terms of lifestyle, were inferred in a Bayesian analysis  (see details in \hyperref[sec:MM-5]{Materials and methods})} \label{figure:Lifecycle_Events_per_genomic_structures}.
\end{figure}

\subsection{Endogenizations of dsDNA viruses are more frequent in endoparasitoid species }

Next, we sought to characterize the factors that could explain the patterns of endogenization events inferred (\figurename{\ref{figure:Event_phylogeny}}). To this end, for each viral genomic structure, we assessed whether  endogenization events were evenly distributed among the three wasp lifestyles, taking into account their respective frequencies in the dataset. No significant departure from the null hypothesis was detected for endogenization involving ssDNA, dsRNA or ssRNA viruses (Fisher exact test p-values BH corrected $>$ 0.05). On the contrary, we detected a highly significant enrichment of dsDNA viruses endogenization events in endoparasitoid species, and conversely a deficiency in free-living and ectoparasitoid species (corrected p-value = 7.8e-04, \figurename{ \ref{figure:Lifecycle_Events_per_genomic_structures}-A}). 

To further test the apparent correlation between Hymenoptera lifestyle and the rate of endogenization events, we inferred ancestral lifestyles along the phylogeny using a Bayesian model (see details in  \hyperref[sec:MM-12]{Materials and methods}). We then constructed a generalized linear model where the dependent variable is the number of endogenization events inferred on each branch, while branch length and lifestyle are the explanatory variables (see details in \hyperref[sec:MM-5]{Materials and methods}). Branch length was included as an additive effect to remove the expected effect of time on the number of endogenization events, thus allowing the decomposition of the remaining variance according to the lifestyle (free-living, ectoparasitoid or endoparasitoid).

We first tested whether the rate of endogenization events deriving from any virus (that is, regardless of  their genomic structures) was structured by lifestyles, and found no significant effect (\figurename{ \ref{figure:All_violin_plots}-A left side}). 
We then split the dataset according to the genomic structure of the donor viruses. For RNA or ssDNA virus, the analysis did not reveal evidence of a correlation between wasps' lifestyles and the rate of endogenization events (\figurename{\ref{figure:All_violin_plots}}-G,I and K). On the contrary, in the case of dsDNA viruses, we found a highly significant effect of the wasp lifestyle: endogenization rates appear to be  2.47 times higher in endoparasitoids than in free-living species (89\% CI [1.56-3.56], \figurename{\ref{figure:dsDNA_violin_plot}}-A). The corresponding probability of direction (pd, an index representing the confidence in the direction of an effect) was equal to 99.9\%. In contrast, ectoparasitoids did not differ from free-living species (\figurename{\ref{figure:dsDNA_violin_plot}}-A). Accordingly, more than 98\% of the MCMC iterations led to a higher coefficient value for endoparasitoids than for ectoparasitoids (so-called $P_{MCMC}$ in \figurename{\ref{figure:dsDNA_violin_plot}}-A). This effect was consistently found using high confidence scaffolds only (A-ranked scaffolds, \figurename{\ref{figure:All_violin_plots}}-C right side). We also carried out the same analysis without the 4 domestication cases previously mentioned in the literature (because including them in our data set could have skewed the results) and reached the same conclusion (\figurename{\ref{figure:All_violin_plots}}-E right and left sides). Overall, these results show that dsDNA viruses are more often endogenized in endoparasitoids than in  free-living and ectoparasitoid species.


\begin{figure}[!htbp]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/dsDNA_violin_plots.pdf}
\caption[Paper1:Violin plot GLM coefficients distribution of dsDNA Events and dEvents]{\textbf{Endogenization and domestication of dsDNA viruses are more frequent in endoparasitoid species}. Violin plots represent the posterior distribution of the coefficients obtained under the different GLM models (after exponential transformation to obtain a rate relative to free-living species). The coefficients are derived from 1000 independent GLM models, where 1000 probable scenarios of ancestral states at nodes were sampled randomly among the MCMC iterations (see details in  \hyperref[sec:MM-12]{Materials and methods}). Branches from nodes older than 160 million years were removed from the dataset. The \%pd is the probability of direction and indicates the proportion of the posterior distribution where the coefficients have the same sign as the median coefficient. $P_{MCMC}$ indicates the proportion of MCMC iterations where the coefficient obtained for endoparasitoid species is higher than for ectoparasitoid species. All statistical summaries of the Bayesian GLM models can be found on the GitHub file: \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/Lifestyle_statistical_analysis_results.xlsx}{Lifestyle\_statistical\_analysis\_results.xlsx}.}
\label{figure:dsDNA_violin_plot}
\end{figure}

\subsection{Domestications of dsDNA viruses are most prevalent in endoparasitoid species}

We then investigated whether lifestyles may explain the abundance of domestication events. A simple Fisher's exact test approach revealed an enrichment in endoparasitoid species of domestication events involving dsDNA viruses (Benjamini-Hochberg adjusted p-value = 1.8e-03), whereas no deviation from the null hypothesis was detected for the other viral genomic structures (\figurename{ \ref{figure:Lifecycle_Events_per_genomic_structures}-B}).

We built upon the generalized linear models described above, in a Bayesian framework, to test whether lifestyle could also be a factor explaining the propensity of Hymenoptera to domesticate (and not simply endogenize) viral genes (see details in \hyperref[sec:MM-5]{Materials and methods}). We found that, domestication of dsDNA viruses are 3.68 times more abundant in endoparasitoids than  in with free-living species (89\% CI [1.72-6.17], pd =99.9\%, \figurename{\ref{figure:dsDNA_violin_plot}}-B). This effect was also detected when only high confidence candidates were considered (\figurename{\ref{figure:All_violin_plots}}-D right side), or if we removed the four known cases of domestication (\figurename{\ref{figure:All_violin_plots}}-F left and right side). In other viral categories, no convincing effect of the wasp lifestyle was detected (all pd$<$99\%) (\figurename{ \ref{figure:All_violin_plots}-H and J}) except
for a higher rate of domestication of ssRNA viruses in free-living than in ectoparasitoid species (\figurename{ \ref{figure:All_violin_plots}-L})

Two non mutually exclusive hypotheses may explain the high frequency of dsDNA virus domestication in endoparasitoids. First, it may simply stem from the higher rate of endogenization outlined above: a higher rate of entry would overall  translate into a higher rate of domestication. Second, it may result more specifically from differences in the rate at which viral elements are domesticated after being endogenized. To disentangle these hypotheses, we built a binomial logistic regression model in a Bayesian framework, focusing on events involving dsDNA viruses, and specifying the number of domesticated events \textit{relative} to the total number of endogenization events inferred. By controlling for the endogenization input (the denominator), these binomial models make it possible to test whether the probability of domestication  after endogenization of dsDNA viruses is correlated with the lifestyle.

Based on this analysis, the probability that an endogenization event will lead to a domestication event is not significantly different between endoparasitoids and freeliving species (\figurename{\ref{figure:dsDNA_violin_plots_corrected_rates}}-A, pd=89.18\%). However, the probability of domestication was found to be significantly higher in endoparasitoids than in ectoparasitoids (\figurename{\ref{figure:dsDNA_violin_plots_corrected_rates}}-A, $P_{MCMC}$=99.81\%). The same trend was observed if we focused on high confidence scaffolds and/or if we removed the 4 known controls from the dataset (\figurename{\ref{figure:dsDNA_violin_plots_corrected_rates}}-B,C and D, pd $<$ 86\%).

Together, these findings show that the endoparasitoid lifestyle is associated with an increased rate of dsDNA viruses endogenization. Endoparasitoids are also characterized by an elevated frequency of domestication events that does not appear to be explained by an elevated rate of post-endogenization domestication. 

\subsection{New remarkable cases of endogenization and domestication}

Here, we describe in more details specific cases identified by our pipeline. We found a massive entry of genes from dsDNA viruses in an undescribed species belonging to the Campopleginae subfamily ("Campopleginae sp" in \figurename{ \ref{figure:Event_phylogeny}}). In Ophioniformes (a clade that includes Campopleginae), two lineages have previously been shown to host domesticated viruses (the Campopleginae species \textit{Hyposoter didymator} \citep{volkoff_analysis_2010-1}, and the Banchinae species \textit{Glypta fumiferanae} \citep{beliveau_genomic_2015-1}). It has been advocated that these so-called ichnoviruses found in \textit{Hyposoter didymator} and \textit{Glypta fumiferanae} may derive from the same endogenization event \citep{beliveau_genomic_2015-1}. In our unknown Campopleginae species, we identified homologs of 35 out of the 40 ichnovirus genes present in the genome of \textit{H. didymator} (so-called IVSPER genes, \citep{volkoff_analysis_2010}). Those genes show conserved synteny in the two species (\figurename{ \ref{figure:Campoplegniae_synteny}}), strongly suggesting that they derive from the same endogenization event. However, our analysis did not identify viral homologs in the two Ophioninae and Cremastinae sufamilies, that are internal to the clade including Campopleginae and Banchinae wasps. This result argues against the view of a single event at the root of Ophioniformes, and thus supports the alternative view \citep{burke_endogenization_2020-1} that the so-called IVSPER genes  in the Campopleginae and Banchinae subfamilies stem from independent events, despite their striking structural similarities (see \figurename{\ref{figure:Ophioniformes_phylogeny}} for illustration). We found no trace of the previously suggested remnants of ichnoviruses in the related species \textit{Venturia canescens}, whereas the presence of nudiviral genes in this species was confirmed \citep{pichon_recurrent_2015}.\\

\begin{figure}[!htbp]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/dsDNA_phylogeny.pdf}
\caption[Paper1:Phylogenetic relationships among endogenized and "free-living" dsDNA viruses]{\textbf{Phylogenetic relationships among endogenized and "free-living" dsDNA viruses}. Specifically, this figure shows the relationships between \textit{Naldaviricetes} double-stranded DNA viruses and EVEs from hymenopteran species, where at least 3 endogenization events were found. This tree was computed using maximum likelihood in Iqtree (v2) from a 38,293 long protein alignment based on the concatenation of 142 viral genes. Confidence scores (aLRT\%/ultra-bootstrap support\%) are shown at each node. The scale bar indicates the average number of amino acid substitutions per site. Previously known EVEs  are in white, those from the present study in orange, and leaves inferred as free-living viruses are in red.  All the best partitioned models can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/dsDNA_phylogeny_best_ML_partitions.nxs}{dsDNA\_phylogeny\_best\_ML\_partitions.nxs}. All free-living dsDNA viruses used in this phylogeny were obtained from published complete viral genomes. More details on the phylogenetic inference can be found in methods.}
\label{figure:dsDNA_phylogeny}
\end{figure}

We found 5 new cases of endogenization involving multiple EVEs from dsDNA viruses belonging to \textit{Nudiviridae}, LbFV-like and AmFV-like families. 

Two of them involve parasitoid species, i.e. \textit{Platygaster orseoliae} and an \textit{Aprostocetus} species. For \textit{Aprostocetus}, we detected 6 EVEs related to nudiviruses branching between the Chalcidoidea and the Diaprioidea superfamilies (\figurename{\ref{figure:Multiple_EVEs_COV_GC}}). Among these EVEs we found four with an annotation : \textit{lef-4}, \textit{Ac68/pif-6}, GrBNV\_gp19/60/61-like proteins,  and a rep-like protein. In the absence of closely related sequences or RNA seq data, we cannot investigate if these elements have been domesticated. The \textit{P. orseoliae} case involve the recently characterized putative family of filamentous viruses \citep{lepetit_genome_2017}. The free-living LbFV virus is the only representative of this putative family and has been identified as a source of adaptive genes in \textit{Leptopilina} wasps that parasitize \textit{Drosophila} flies, with 13 virally-derived genes involved in the production of VLPs protecting the wasp's eggs from encapsulation (Di Giovanni et al., 2020). In \textit{P. orseoliae}, 15 genes homologous to LbFV were detected (out of 108 ORFs in the LbFV genome; median E-value = 9.39e-21 [min = 2.617e-76, max = 4.225e-08], \figurename{ \ref{figure:Porseoliae_endogenization_events_gene_plot}}-A). Among these 15 genes, 5 were also endogenized in \textit{Leptopilina} species (named LbFV\_ORF58:DNApol, LbFV\_ORF78, LbFV\_ORF60:LCAT, LbFV\_ORF107 and LbFV\_ORF85) (Di Giovanni et al., 2020). Assuming the ancestral donor virus contained the same 108 genes as LbFV, the number of shared genes in these two independent domestication events is higher than expected by chance (one-sided binomial test: x = 5, n = 15, p = 13/108, p= 0.02682), suggesting  that similar functions could have been retained in both lineages (see additional information in \hyperref[sec:SI-2]{supplemental information}). Notably, we also found within the \textit{P. orseoliae} assembly a new "free-living" virus among scaffolds noted as F or X, related to LbFV, that we propose to call PoFV (Platygaster orseoliae filamentous virus) (see fig. \ref{figure:dsDNA_phylogeny} and  \hyperref[sec:SI-2]{supplemental information} for details). This virus is the closest relative to the EVEs found in \textit{P. orseoliae} and is composed of 136 ORFs \figurename{\ref{figure:Porseoliae_endogenization_events_gene_plot}}-B). Using this putative whole genome viral sequence to search for homologous genes in the \textit{P. orseoliae} genome, we were able to detect a total of 139 convincing EVEs (corresponding to 89/136 PoFV ORFs). 44/89 of these EVEs presented signs of domestication using paralogs, with \textit{dN/dS} significantly lower than 1 (see details in  \hyperref[sec:SI-2]{supplemental information}). Although functional studies are clearly needed to confirm that these virus-derived genes are involved in the production of VLPs as in \textit{Leptopilina} (Di Giovanni et al., 2020), we see Platygaster orseoliae endogenous viral elements (PoEFVs) as good candidates for viral domestication, which could possibly be involved in counteracting the immune system of its dipteran host (from the Cecidomyiidae family \citep{buhl_new_2016}). To our knowledge, this is the first report of a massive viral endogenization and putative domestication within the \textitit{Platygastroidea} superfamily.

The other three cases involved ant species : \textit{Harpegnathos saltator} (EsEFV) 
(12EVEs and 6dEVEs), \textit{Pseudomyrmex gracilis} (PgEFV) (9EVEs and 1dEVE), \textit{Aphaenogaster picea} (ApEFV) (7EVEs). These endogenized elements are related to a poorly characterized family of filamentous viruses denoted AmFV \citep{hartmann_dynamics_2015,yang_genomics_2022}. In \textit{H. saltator}, 9 genes deriving from an AmFV-like virus were detected (including 3 genes that have been previously identified by \citep{flynn_assessing_2019}). Intriguingly, all these genes presented  numerous paralogs within the genomes (135 in total) (\figurename{\ref{figure:Multiple_new_endogenization_events_gene_plot}} and \figurename{\ref{figure:Multiple_EVEs_COV_GC}}), with 22 copies for AmFV\_0062 (\textit{pif-1}), 18 for AmFV\_0102 (\textit{pif-2}), 51 for AmFV\_0090 (\textit{pif-3}), 24 for AmFV\_0044 (\textit{integrase}), 13 for AmFV\_0079 (\textit{p74}), 5 for AmFV\_0047 (\textit{RNA polymerase}), 19 for AmFV\_0126 (Unknown), 23 for AmFV\_0168 (Unknown) and  7 for AmFV\_0154 (Unknown). Most paralogs were found in scaffolds exceeding the expected size for any virus sequence (min = 23,726bp, mean = 326,262bp, max = 2,693,376bp). In addition, all scaffolds do include transposable elements and eukaryotic genes making them undoubtedly endogenized. Accordingly, our pipeline attributed the highest confidence index A for 104 of them (out of 135). The \textit{P.gracilis} genome revealed 9 EVEs, including homologs of \textit{pif-1}, \textit{pif-3}, \textit{RNA polymerase}, \textit{ac81}, \textit{integrase} and \textit{odv-e56} (\figurename{\ref{figure:Multiple_new_endogenization_events_gene_plot}}). Notably, one of the 9 EVEs (AmFV\_059, of unknown function) shows both a \textit{dN/dS} $<$ 1 (mean= 0.1747, p-value 5.877e-02), and a very high TPM value (362836 TPM from whole body tissues). Finally, in \textit{A. picea}, 7 EVEs were detected, including homologs of \textit{pif-1}, \textit{pif-3}, \textit{integrase}, \textit{odv-e56} and \textit{p74} (\figurename{ \ref{figure:Multiple_new_endogenization_events_gene_plot}}). No raw reads data were available for this species, precluding coverage-based inferences. Since there were neither orthologs or paralogs for these genes to compute \textit{dN/dS} analyses, nor transcriptomic data, it was not possible to infer their domestication status. At this stage it is thus not possible to conclude as to the functions of these genes in \textit{H. saltator}, \textit{P. gracilis} and \textit{A. picea}, but this surely deserves further attention.

 
\section{Discussion}

All kinds of viruses can integrate arthropod genomes, although the mechanisms underlying these phenomena remain unclear \citep{katzourakis_endogenous_2010, gilbert_diversity_2022}. Prior to the present analysis, 28 viral families had been described as involved in endogenization in arthropods \citep{gilbert_diversity_2022}. Our study of Hymenopteran genomes further revealed the ubiquity of this phenomenon, with at least 40 viral families (or family-like clades) involved. Of the 1,261 EVEs found, the average identity with the closest known viral proteins was 36.32\% [min = 15.7\%, max = 99.1\%]. Although this large overall divergence suggests ancient events, it does not exclude the possibility that some of the integrations are recent, because free-living viral relatives of the true donors may be unknown or extinct \citep{junglen_virus_2013}.

In the following section, we will first discuss why double-stranded DNA viruses, in comparisons with other viral genomic structures (ssDNA, dsRNA, ssRNA), are more often endogenized than expected. We will then discuss hypotheses that could explain why we found a higher rate of endogenization of dsDNA viruses among  endoparasitoids compared to ectoparasitoids or free-living hymenopterans, which also translates into more frequent events of domestications.\\

\textbf{\\dsDNA viruses are more frequently involved in endogenization than expected by chance}
\\Despite the observations that all viral genomic structures can be involved in endogenization, we clearly identified differences in their propensity to do so. Based on a comparison between the respective proportions of the various viral categories in the inferred endogenization events and in public databases, we found that dsDNA viruses are much more represented than expected, while ssRNA viruses are under-represented (\figurename{ \ref{figure:Genomic_structure_and_families_Events_distributions}-A}). We acknowledge that current knowledge on the actual diversity of free-living viruses (as approximated through the NCBI taxonomy database) remains incomplete, but the strength of the effect reported here makes this conclusion rather robust to variations in the null distribution. On the basis of current knowledge, RNA viruses, and in particular ssRNA viruses, appear to be much more diversified and prevalent than DNA viruses in insects. We note that viral-metagenomic studies often focus either on DNA or RNA viruses, and as such do not provide an accurate and unbiased picture of the extent viral diversity. To gain insights on this topic, we may thus focus on model systems where long-lasting research efforts have likely produced a more reliable picture. The Honeybee \textit{Apis mellifera} is probably the most studied of all Hymenopteran species. In honeybees, the great majority of known viruses belongs to the RNA world \citep{chen_honey_2007}, with very few exceptions \citep{yang_genomics_2022}. Similarly, until 2015, only RNA viruses were known to infect the fruit fly \textit{Drosophila melanogaster}, despite the extensive research conducted on this model system \citep{webster_discovery_2015}. A very limited set of DNA viruses has now been described from this species \citep{wallace_discovery_2021} but clearly, RNA viruses dominate the \textit{Drosophila} viral community, both in terms of diversity and prevalence. In support of this view, recent studies revealed the very elevated absolute diversity of RNA viruses. For instance, a survey of 600 insect transcriptomes recovered more than 1,213 RNA viruses belonging to 40 different families \citep{wu_abundant_2020-1}. Although, obviously, this study does not inform on the diversity of DNA viruses, it shows that the RNA virome of insects is both prevalent (e.g., in this study, 15\% of all insects were infected by a single Mononegales-like virus) and extremely diversified \citep{wu_abundant_2020-1}. Actually, this view appears to hold at the larger scale of eukaryotes \citep{koonin_origins_2015}. Taking into account this patent abundance of ssRNA viruses in insects, our study indicates they are by far less frequently endogenized than their dsDNA counterparts in hymenopterans. Notably, a similar trend was recently reported in a study including a diverse set of eukaryotes \citep{irwin_systematic_2022}.

Most of the major endogenization events characterized so far in hymenopterans involve dsDNA viruses from the \textit{Nudiviridae} family \citep{cheng_nudivirus_2020, burke_common_2019,pichon_recurrent_2015,bezier_polydnaviruses_2009-1, cheng_brown_2014, zhang_chalcid_2020, gilbert_diversity_2022}. Our study further confirms that this viral family represents a major source of exogenous and sometimes adaptive genes for Hymenoptera. Indeed, 28 new independent endogenization events involve this family, among which 9 are shared by at least two related species (\figurename{ \ref{figure:Genomic_structure_and_families_Events_distributions}-B}, \figurename{\ref{figure:Event_phylogeny}}). 
The major contribution of nudiviruses to endogenization may be explained by their wide host range in arthropods \citep{wang_nudivirus_2007}.  Their nuclear replication constitutes another plausible explanatory factor \citep{velamoor_visualizing_2020}, since it may facilitate contact with host DNA. In addition, their tropism for gonads may favor the endogenization in germinal cells \citep{burand_sexually_2009}. In fact, nuclear replication is a feature shared by nearly all families of dsDNA viruses found in our analysis : \textit{Baculoviridae}, \textit{Iridoviridae}, \textit{Phycodnaviridae}, \textit{Nimaviridae} \textit{Caulimoviridae}, \textit{Herpesviridae}, \textit{Asfaviridae} (at early times) \citep{schmid_dna_2014, harrison_ictv_2020, international_committee_on_taxonomy_of_viruses_virus_2012, teycheney_ictv_2020,verbruggen_molecular_2016}, Apis-filamentous-like \citep{clark_filamentous_1978} and LbFV-like families \citep{varaldi_artifical_2006} (the \textit{Poxviridae} viruses, that replicate in the cytoplasm, are thus the only exception). In contrast, most RNA viruses replicate in the cytoplasm. Nuclear replication may thus constitute a general explanation for the elevated propensity of DNA viruses to endogenization. Additionally, we may expect that a DNA molecule, rather than an RNA molecule, is more likely to integrate the insect genome, because the latter requires reverse transcription before possible endogenization.

The \textit{Poxviridae} case indicates that cytoplasmic replication does not necessarily impede endogenization. These viruses do not require nuclear localization to propagate \citep{moss_poxvirus_2013,schmid_dna_2014} and were nevertheless found to be involved in many endogenization events (n=28). A similar pattern was observed in a recent study focusing on ant genomes \citep{flynn_assessing_2019}). Within Poxviridae, entomopoxviruses were particularly involved in endogenization events (n=18) with four cases of EVEs shared between several closely related species (\figurename{ \ref{figure:Genomic_structure_and_families_Events_distributions}-B}).\\

\textbf{\\Factors behind variations in endogenization and domestication rates }
\\Several recent studies have uncovered abundant EVEs in insect genomes \citep{flynn_assessing_2019, ter_horst_endogenous_2019, russo_novel_2019}, with huge variation in abundance between species.  For instance, in their analysis based on 48 arthropod genomes, \citep{ter_horst_endogenous_2019} found that the number of EVEs ranged between 0 and 502. Although insect genome size and assembly quality may partly explain this variation \citep{gilbert_diversity_2022}, the underlying biological factors are generally unknown. In this study, we tested the hypothesis that the insect lifestyle may influence both the endogenization and domestication rates. We used a Bayesian approach to reconstruct ancestral states throughout the phylogeny of Hymenoptera, thus accounting for uncertainty, and found that endoparasitoidism, in comparison with other lifestyles, tends to promote dsDNA viral endogenization. Notably, this conclusion was not the artefactual consequence of differences in genome assembly quality. In fact, the quality of genome assemblies was correlated with the lifestyle in our data set, but the genomes of endoparasitoid species were generally less well assembled than those of free-living species. If anything, this difference should reduce the power for detecting endogenization events in endoparasitoids, where our analysis detected an excess of such events. Our estimate of the effect sizes (with 2.47 times more endogenization events in endoparasitoids than in  free-living species) should thus be seen as conservative. 
Why do endoparasitoid wasps tend to undergo more endogenization than others? We initially had in mind two non-exclusive hypotheses that remain plausible explanations for the observed pattern. First, endoparasitoids may be more intensively exposed to viruses. In addition, or alternatively, endoparasitoids may have a higher propensity to endogenize and retain viral genes. 

Several factors come in support of the first hypothesis. Endoparasitoid larvae grow by definition inside their host's body, and such a close interaction implies that any endoparasitoid individual will also be interacting with its host's viruses. Accordingly, the best studied cases of viral domestication in wasps involve nudiviruses, that are known to replicate in their caterpillar hosts \citep{burke_rapid_2018,pichon_recurrent_2015,bezier_polydnaviruses_2009-1}. Another putatively important factor is the presence of virus in the venoms that parasitoids inject into hosts together with their eggs. These are known to protect the offspring against the host's immune response, and to manipulate the host physiology \citep{asgari_venom_2011} but this feature could favor the subsequent spread of viruses in wasp populations: by colonizing the venom-producing tissues (venom gland or calyx, depending on the species biology) viruses may secure an effective pseudo-vertical transmission and  thus maintain themselves efficiently in wasp populations. Numerous endoparasitoid viruses benefit from such pseudo-vertical transmission  \citep{varaldi_artifical_2006,renault_cypovirus_2003}, including some whose relatives have been endogenized and domesticated by endoparasitoids (Di Giovanni et al., 2020). The presence of viruses within venoms may also facilitate horizontal transmission between conspecifics in the case of superparasitism, as observed in the \textit{Drosophila} parasitoid \textit{Leptopilina boulardi} \citep{varaldi_infectious_2003}. Although this effect may also be at play for some ecto-parasitoid species \citep{zhang_novel_2021}, we expect it to be more pronounced for endo-parasitoid species since they have a closer interaction from the inside of their hosts. 
Generally, endoparasitoids may thus carry a higher load of non-integrated viruses than other hymenopterans. However, if this effect is at play, we expect to have an "endoparasitoid" effect for all viruses, whatever their genomic structure. For instance, we would expect such an effect to be detected for ssRNA viruses, which are involved in the greatest number of endogenization events (\figurename{\ref{figure:Genomic_structure_and_families_Events_distributions}}-A).  This was not the case, since only dsDNA viruses were more frequently endogenized in endoparasitoids. Thus, we argue that this hypothesis is unlikely to explain the observed pattern.

The second hypothesis posits that endoparasitoids are more frequently selected for retaining  virally-derived genes than ectoparasitoid or free-living hymenopterans. In our analysis, domestication events are most frequently observed in endoparasitoids (over 3 times more frequently than in other hymenopterans). Obviously, this may be at least partly explained by the higher input discussed above (the higher endogenization rate). Yet, once this effect is controlled for, a trend towards a higher rate of domestication remains. More specifically, the likelihood of domestication following endogenization was significantly higher in endoparasitoids than in ectoparasitoids, but was not significantly higher than in free-living species. This latter lack of signiﬁcant difference may be biologically explained if a single domestication event precludes the domestication of additional EVEs, while not affecting the rate of non-adaptive endogenization. This would "dilute" the signal along branches involved in domestication. If this effect is at play, then it reduces considerably the power of our analysis to detect any difference on the rate of domestication between lifestyles. Indeed, in all known cases, only one domesticated virus has been documented, suggesting that further domestications are not beneficial once a viral machinery has been recruited by a wasp lineage.

Whether or not the rate of domestication \textit{per se} is higher in endoparasitoids than in other hymenopterans, the selective advantages brought by these viral genes in endoparasitoids should be discussed. It has been demonstrated in a few model systems that EVEs may confer antiviral immunity against related "free-living" viruses via the piRNA pathway  \citep{suzuki_non-retroviral_2020,whitfield_diversity_2017}. Yet, to our knowledge, such an effect has only been demonstrated against RNA viruses, so that it would not explain the excess of DNA viruses documented here. Furthermore, the sequence identities with known viral sequences, which is needed for this mechanism to work, is low in our dataset. Accordingly, previous work revealed that EVE-derived piRNAs studied in 48 arthropod species were also probably too divergent to induce an efficient antiviral response \citep{ter_horst_endogenous_2019}. At that stage, the ability of EVEs to generate PIWI-interacting RNAs that play a functional role in antiviral immunity seems questionable. Further studies involving small RNA sequencing in hymenopterans would be required to shed light on this issue.
Protection of the eggs and larvae against the host immune system is recognized as an important trait, where EVEs play a critical role. Because of their peculiar lifestyle, endoparasitoids are all targeted by the host immune system, a matter of life or death to which other hymenopterans are not exposed to. Several cases of endogenization and domestication in endoparasitoids, all involving dsDNA viruses, are thought to be related to this particular selective pressure \citep{bezier_polydnaviruses_2009-1,volkoff_analysis_2010,pichon_recurrent_2015,burke_common_2019,di_giovanni_behavior-manipulating_2020}). The parasitoids appear to have co-opted the viral fusogenic property to address their own proteins (VLPs) or DNA fragments (polydnaviruses) to host immune cells, thereby canceling the host cellular immune response. The above-hypothesized high exposure of endoparasitoids to viruses, together with this unique selective pressure, may act in concert to produce the pattern documented here: a strong input, that is, a diverse set of putative genetic novelties, combined with a strong selective pressure for retaining some of them. The observed excess of dsDNA viruses  may be an indication  that these viruses display a better potential for providing adaptive material in this context. In the cases of polydnaviruses (found in some Braconidae and some Campopleginae), it appears that one way to efficiently deliver virulence factors to the host cell is by addressing DNA circles that ultimately integrate into the host immune cells and get expressed \citep{chevignon_functional_2014, chevignon_cotesia_2018}. The DNA which is packed into the mature particles typically encodes virulence proteins deriving from the wasp \citep{espagne_genome_2004}. This means that, at least for these cases, the viral system should be able to pack DNA, which is most likely a feature that DNA viruses may provide. Such an argument does not hold in the VLP systems, where only proteins are packed in viral particles, and it is unclear why EVEs deriving from dsDNA viruses would be more able to fulfill such a function. Here other features of dsDNA viruses come into mind as possibly important factors: their large genome size, and their large capsids and envelopes \citep{chaudhari_scaling_2021}. These may predispose dsDNA viruses to be domesticated, since abundant quantities of venoms have to be transmitted in order to efficiently suppress the host immune response.


\section{Conclusion}

Our analysis has revealed a large set of new virally-derived genes in Hymenoptera genomes. Those genes were deriving from viruses with any genomic structures, although dsDNA viruses were disproportionately involved in endogenization and domestication. 
Importantly, our analysis revealed that endogenization rate and the absolute number of domestication events involving dsDNA viruses was increased for endoparasitoids compared to other lifestyles. 
Among the new cases of endogenization and domestication, we uncovered new events revealing common features with previously known cases of viral domestication by endoparasitoids, such as in the Platygastroidea \textit{Platygaster orseoliae}. This is to our knowledge the first case reported in the superfamily Platygastroideae, thus extending the diversity of Hymenoptera concerned by viral domestication. We propose that the higher rate of endogenization and higher number of domestication events in endoparasitoids is a consequence of the extreme selective pressure exerted by the host immune system on endoparasitoids. This extreme selective pressure may select endoparasitoids for retaining a viral machinery that could help them address virulence factors to their hosts. We expect this process to be widespread among insect species sharing the same lifestyle.

\section{Material and methods}


\subsection{Genome sampling, assembly correction and assembly quality}
\label{sec:MM-1}

A bioinformatic pipeline mixing sequence homology search, phylogeny, genomic environment, and selective pressure analysis was built to search for viral endogenization and domestication events in Hymenoptera genomes. We used 133 genome assemblies in total, of which 101 were available on public repositories (NCBI and BIPPA databases) and 32 were produced by our laboratory (all SRA reads and assemblies available under the NCBI submission ID : SUB11373855). Concerning the last 32 samples, DNA was extracted on single individuals (usually one female) or a mix of individuals when the specimens were too small using Macherey-Nagel extraction kit, the DNA was then used to construct a true seq nano Illumina library at Genotoul platform (Toulouse, France). The sequences were generated from HiSeq 2500 or HiSeq 3000 machines (15Gb/sample). The paired-end reads were then quality trimmed using fastqmcf (-q15 --qual-mean 30 -D150, \href{https://GitHub.com/ExpressionAnalysis/ea-utils}{GitHub}) and assembled using IDBA-UD \citep{peng_idba-ud_2012}. All sample information can be found on the GitHub file: \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/All_sample_informations.txt}{All\_sample\_informations.txt} and is available under the NCBI Biosample number : SUB11338872.

The size of the 133 assemblies ranged from 106.14mb to 2102.30mb. We kept only genome assemblies containing at least 70\% non-missing BUSCO genes (124/133 genomes, \citep{simao_busco_2015}) (all genome information can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/Assembly_genome_informations.csv}{Assembly\_genome\_informations.txt})). In addition, when the raw reads were available, we used the MEC pipeline \citep{wu_mec_2020} to correct possible assembly errors. Although some genomes were highly fragmented (such as the 32 genomes we generated since they were obtained using short reads only), the N50 values (min: 3542bp) were equal to or larger than the expected sizes of genes known to be endogenized and domesticated (min known domesticated EVE : 165bp) indicating that most of the putative EVEs should be detected entirely.

Out of the 32 samples sequenced by our laboratory for this study, one (corresponding to \textit{Platygaster orseoliae}) gave unexpected results. After assembly and BUSCO analysis, two sets of contigs were identified:  one with only 4X coverage on average, and one with 33X on average. The phylogeny of these different BUSCOs gene sets showed that the low-coverage scaffolds likely belong to an early diverging lineage of Chalcidoidea  (\figurename{\ref{figure:Event_phylogeny}}), whereas the 33x scaffolds belong to the target species \textit{P. orseoliae}. This result suggests that the pool of 10 individuals used for sequencing was likely a mix of two species. A phylogenetic study based on Ultra Conserved Elements (UCEs) obtained from several species of Chalcidoidea \citep{rasplus_first_2020, cruaud_optimized_2019} allowed to identify the unknown species as sister to  \textit{Aprostocetus sp}  (Eulophidae) (see details in the \hyperref[sec:SI-2]{supporting information} and \figurename{\ref{figure:UCEs}}). In the figures and tables, the name putative\_ \textit{Aprostocetus}\_sp was consequently assigned to the unknown sample. However, since the lifestyle and identity of this species are uncertain, we did not include the corresponding scaffolds in the main analysis. The scaffolds belonging to this putative\_\textit{Aprostocetus}\_sp. (i.e : all scaffolds with a mean coverage $<$ 10X) were removed from the \textit{P. orseoliae} assembly file hosted in NCBI.  

\subsection{Pipeline outline}
\label{sec:MM-2}\\
EVEs were identified from the 124 Hymenoptera assemblies using a sequence-homology approach against a comprehensive viral protein database (including all categories of viruses : ssDNA, dsDNA, dsRNA and ssRNA). In order to validate viral endogenization within Hymenoptera genomes, we developed an "endogenization confidence index" ranging from A to X (\figurename{\ref{figure:Bioinformatic_pipeline}}-7). This index takes into consideration the presence of eukaryotic genes and/or transposable elements around candidate loci, and scaffolds coverage information (coverage for a valid candidate should be similar to that found in BUSCO containing scaffolds). Finally, the pipeline also included an assessment of the evolutionary history and of the selective regime shaping the candidates (based on \textit{dN/dS} and/or expression data).

\subsection{Hymenoptera phylogeny}
\label{sec:MM-3}

The phylogenetic reconstruction of the 124 Hymenoptera species was performed based on a concatenation of the 375 BUSCO proteins. The analysis was conducted by maximum likelihood via Iqtree2 \citep{minh_iq-tree_2020} selecting the best model \citep{kalyaanamoorthy_modelfinder_2017}. The tree was rooted via two species of the Coleoptera order (\textit{Anoplophora glabripennis} and \textit{Tribolium castaneum}). Bootstrap scores were evaluated using the UFboot approach \citep{hoang_ufboot2_2018}. The results found were consistent with a previous, more comprehensive study \citep{peters_evolutionary_2017}. 

subsection{Search for viral homology}
\label{sec:MM-4}

We collected all protein sequences available in NCBI virus database \citep{hatcher_virus_2017}, removing phage and polydnavirus (virulence genes from wasp origin found within PDVs) sequences. This database contained 849,970 viral protein sequences (download date : 10/10/2019), to which the 40 putative viral proteins encoded by the \textit{Hyposoter didymator} genome were added (so-called IVSPER sequences, \citep{volkoff_analysis_2010-1}). The sequence homology search was performed with a BlastX equivalent implemented in Mmseqs2 \citep{steinegger_mmseqs2_2017}) using each genome assembly as queries and the viral proteins collected as database. The result gave a total of 81,953,678 viral hits (max E-value 5e-04 with an average of 660,916 hits per genomes). We kept only candidates with a percentage coverage of the viral protein $>$= 30\%, an identity score $>$= 20\% and an E-value score $<$ 5e-04 (\figurename{ \ref{figure:Bioinformatic_pipeline}}-1). The threshold parameters were optimized to maximize the detection of the 13 endogenous viral sequences within the genus \textit{Leptopilina} (Di Giovanni et al., 2020). Once all the viral hits were recovered, we formed putative EVEs loci (n=238,108) corresponding to the overlap of several viral hits on the same scaffold using the GenomicRanges R package \citep{lawrence_software_2013} (\figurename{ \ref{figure:Bioinformatic_pipeline}}-2).
To remove false positives corresponding to eukaryotic genes rather than viral genes, we then performed another generalist sequence homology search against the Nr database (downloaded the 09/11/20) using mmseqs2 search (-s 7.5, E-value max = 0.0001) (\figurename{ \ref{figure:Bioinformatic_pipeline}}-3). We did not select our candidate based on the best hit, since it does not necessarily reflect the true phylogenetic proximity. Instead, candidates with more than 25 hits with either eukaryotic non-hymenoptera species or prokaryotic species were removed, except if they also had hits with at least 10 different virus species (bits $>$= 50). We chose to eliminate Hymenoptera hits from the database because if a real endogenization event concerns both one of the 124 species of our dataset and some species in the NCBI database, then an apparent "Hymenoptera" hit will be detected, possibly leading to its (unfair) elimination.
Since viral diversity is  poorly known, we also kept sequences with even one single viral hit, as long as it did not have more than 5 eukaryotic  or prokaryotic hits. Using these filtering criteria we removed a total of 234,036/238,108 (98,3\%) candidate loci leaving 4,072 candidates with convincing homology to viral proteins. Note that among these loci a certain proportion actually corresponded to non-endogenized "free-living" viruses. To study the evolutionary history of these candidate EVEs, we then performed a general protein clustering of all the candidates and the NCBI viral proteins (\figurename{ \ref{figure:Bioinformatic_pipeline}}-4, Mmseqs cluster; thresholds : E-value 0.0001, cov\% 30, options : --cluster-mode 1 --cov-mode 0 --cluster-reassign --single-step-clustering \citep{steinegger_clustering_2018}.

We eliminated from the dataset the chuviral glycoproteins that have been captured by LTR retrotransposons \citep{dezordi_and_2020}, as these loci have complex histories mostly linked to the transposition activity after endogenization. For this purpose, we systematically searched among the candidates for the presence of TEs within or overlapping with the EVE (see the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/All_TEs_overlapping_with_EVEs.txt}{All\_TEs\_overlapping\_with\_EVEs.txt}). Only one cluster (Cluster4185) was concerned by such a situation (chuviral glycoproteins overlapping to Gypsy/LTRs). It was detected in 89/124 species (1074 total copies, median = 7 copies/species, max = 244, min =1), and was probably similar as the one described in \citep{li_unprecedented_2015}. 

\subsection{Evolutionary history and selection pressure acting on endogenous loci}

\subsection{Arguments for endogenization} 
\label{sec:MM-5}

Among all the candidates for endogenization there were probably false positives that corresponded either to natural contaminants (infecting viruses sequenced at the same time as the eukaryotic genome) or laboratory contaminants (virus accidentally added to the samples). One way to filter these cases was to study (i) the genomic environment (are there other eukaryotic genes or transposable element on the same scaffolds?) and (ii) metrics such as G+C\% (used only for read coverage/GC plots) and scaffold coverage depth around candidate loci (are they the same as scaffolds containing housekeeping genes?). All of these data were used to establish confidence in the endogenization hypothesis, scaled from A to X (\figurename{ \ref{figure:Bioinformatic_pipeline}}-7).

\textbf{(i) Scaffolds sequencing depth (\figurename{ \ref{figure:Bioinformatic_pipeline}}-5) :}
In order to support the hypothesis that a scaffold containing candidate EVEs was part of the Hymenoptera genome, we studied the sequencing depth of the scaffolds. If the sequencing depth of a candidate scaffold was not different from the depth observed in scaffolds containing BUSCO genes, then this scaffold was likely endogenized into the Hymenoptera genome. Hence, when DNA reads were available (\figurename{\ref{figure:Genomes_qualities_and_origins}}), we measured this metric by mapping the reads on the assemblies using hisat2 v 2.2.0 \citep{kim_graph-based_2019}. An empirical p-value was then calculated for each scaffold containing a candidate EVE. To calculate this empirical p-value, we sampled 500 loci of the size of the scaffold of interest within BUSCO scaffolds. These 500 samples represented a null distribution for a scaffold belonging to the Hymenoptera genome. The p-value then corresponded to the proportion of BUSCO depth values that were more extreme than the one observed in the candidate scaffold (two-sided test). We used a threshold of 5\% and a 5\% FDR (multipy python package \citep{puolivali_influence_2020}). 

\textbf{(i) Genomic environment and scaffold size} (\figurename{ \ref{figure:Bioinformatic_pipeline}}-6)
Another way to rule out contaminating scaffolds was to look for the presence of eukaryotic genes and transposable elements in the scaffolds containing candidate EVEs, assuming that their presence in a viral scaffold is unlikely. Indeed, so far, very few viral genomes have been shown to contain transportable elements \citep{miller_virus_1982,gilbert_population_2014,gilbert_continuous_2016,gilbert_viruses_2017,loiseau_wide_2020} because TE insertions are mostly deleterious and are therefore quickly  eliminated by negative selection \citep{gilbert_continuous_2016, gilbert_viruses_2017}. We searched for transposable elements by a BlastX-like approach (implemented in Mmseqs2 search -s 7.5), taking as query the scaffolds of interest and as database the protein sequences of the transposable element (TE) available in RepeatModeler database (RepeatPeps, v2.0.2) \citep{flynn_repeatmodeler2_2020}. We only kept hits with an E-value $<$ 1e-10 and with a query alignment greater than 100aa. We then merged all overlapping hits and 
counted the number of TEs for each scaffold. To find eukaryotic genes within genomes we used Augustus 3.3.3 \citep{stanke_augustus_2004} with BUSCO training and then assigned a taxonomy to these genes via sequence homology with Uniprot/Swissprot database using mmseqs2 search \citep{steinegger_mmseqs2_2017}, and only retained genes assigned to insects.

Accordingly, the scaffolds were scored as follows (\figurename{ \ref{figure:Bioinformatic_pipeline}}-7) : 
\begin{itemize}
    \item A: scaffolds with a corrected coverage p-value $>$ 0.05 and at least one eukaryotic gene and/or one repeat element, 
    \item B: scaffolds with at least one eukaryotic gene and/or one repeat element but no coverage data available, 
    \item C: scaffolds with a corrected coverage p-value $>$ 0.05 and neither eukaryotic gene nor transposable element, 
    \item D: scaffolds with a corrected coverage p-value $<$ 0. 05 and whose coverage value was higher than the average of the scaffolds containing BUSCOs (as it is difficult to imagine that an endogenized scaffold present a lower coverage than expected, whereas a higher coverage could correspond to the presence of repeated elements that inflate the coverage of the scaffold for example) but with at least 5 eukaryotic genes and/or a repeated element (in total), 
    \item E: scaffolds presenting no DNA seq coverage data available and no eukaryotic gene nor transposable element detected, 
    \item F: scaffolds presenting a corrected p-value of coverage $<$ 0.05 and less than 5 eukaryotic genes without any transposable elements; this category may rather correspond to free-living viruses.
    \item X: scaffolds with a corrected p-value < 0.05 and neither eukaryotic gene nor transposable element; This category may rather correspond to free-living viruses.
\end{itemize}


\subsection{Inference of endogenization events}
\label{sec:MM-6}

Because several EVEs may derive from the same endogenization event, we sought to aggregate EVEs into unique events. We aggregated into a single event, firstly (i) all the EVEs present on the same scaffolds, and secondly (ii) all the EVEs that presented the same taxonomic assignment at the level of the viral family.  These two steps were sufficient to aggregate EVEs in the simplest case of events involving only one species (but possibly several EVEs).  

To further characterize the endogenization events including more than one Hymenoptera species, we also relied on phylogenetic inference.  To this end, the protein sequences belonging to each of the clusters (containing both viral proteins and candidate EVEs) were first aligned with clustalo v1.2.4 \citep{sievers_fast_2011} in order to merge possible candidate loci (which may in fact correspond to various HSPs). All loci (=HSPs) within the same scaffold presenting no overlap in the alignment were thus merged, as they probably correspond to multiple HSPs and not duplications.  We then performed a new codon alignment from the augmented sequences in the clusters using the MACSE v2 alignsequence program \citep{ranwez_macse_2018} (\figurename{ \ref{figure:Bioinformatic_pipeline}}-8). This alignment allowed us to obtain a protein and nucleotide codon alignment. We used the protein alignment to infer the phylogeny of each cluster with the program Iqtree2 v2.1.2 \citep{minh_iq-tree_2020} (-m MFP -alrt 1000 (partitioned))(\figurename{ \ref{figure:Bioinformatic_pipeline}}-9).
No trimming was performed at the amino-acid level, since this may result in loss of topology information \citep{tan_current_2015,ranwez_chapter_2020}. However, since it can affect branch length, only codon alignment was trimmed at the protein level via Trimal v1.2 (\figurename{ \ref{figure:Bioinformatic_pipeline}}-10) (-backtrans -automated1) \citep{capella-gutierrez_trimal_2009}. 
We then exploited the information from the cluster phylogenies to form the endogenization events. EVEs potentially deriving from the same event should be supported by the formation of the same well-supported monophyletic clade (bootstrap $>$ 80) both in the gene tree and the Hymenoptera tree (allowing gene losses in 20\% of the species concerned by the monophyletic group). EVEs were possibly aggregated within the same event only if the Hymenoptera belonged to the same family.  (\figurename{ \ref{figure:Bioinformatic_pipeline}}-11). Finally, the clustering of multiple EVEs within the same scaffold in one species was used to aggregate the homologous EVEs found in a related species within the same shared event, even if they were on different scaffolds (\figurename{ \ref{figure:Bioinformatic_pipeline}}). For details, see some canonical examples in \figurename{ \ref{figure:Event_assignation_examples}}. 

For events shared by several species, we were also able to analyze gene synteny around putative EVEs. To do this, we conducted the equivalent of an all vs all TblastX (Mmseqs2 search --search-type 4, max E-value =1e-07) between all the candidate loci within a putative event (deduced from the phylogenetic inference), and then looked for hits (HSPs) between homologous EVEs around the insertions. Because it is possible to find homology between two genomic regions that does not correspond to orthology, for example because of the presence of conserved domains, we had to define a threshold to identify with confidence the orthology signal. We therefore conducted simulations to define this value, based on the well assembled genome of \textit{Cotesia congregata} (GCA\_905319865.3) by simply performing the same all vs all blast analysis against itself (as if the two species considered had the same genome). Based on this, we defined two types of simulated EVEs, (i) independently endogenized EVEs in the genomes of the two "species". This is simply simulated by randomly selecting two different regions in the genomes, and (ii) a shared  simulated EVE that was acquired by their common "ancestor". This is simulated by selecting  the same random genomic location in both "genomes". We then counted the total length of the HSPs found around the simulated insertions all along the corresponding scaffold (i and ii). As the result will obviously depend on scaffold length, we performed these simulations on several scaffold lengths (100000000bp, 10000000bp, 1000000bp, 100000bp and 10000bp). 
We conducted 500 simulations in each scenario, and we measured the cumulative length of homologous sequences by counting the sum of HSPs (bit score $>$ 50). We then defined a threshold for each windows size in order to minimize for the false-positive (FP) and maximize true-positifs (TP)  (thresholds 100000000bp = 172737bp (FP = 0.012, TP= 0.922); 10000000bp = 74262 bp (FP= 0.012, TP=0.878) ; 1000000bp = 21000 bp (FP=0.014, TP=0.28); 100000bp = 1332 bp (FP= 0.012 TP= 0.198) and  10000bp = 180 bp (FP= 0.008, TP= 0.208)). 

Events were linked to viral families based on the closest match information between the viral blastx (GenBank accession number and/or viral protein and/or viral species) and the classification proposed in \citep{shi_redefining_2016}. 


\subsection{Arguments for domestication}}
\label{sec:MM-7}

One way to test for the domestication of an EVEs (dEVEs) was to estimate the ratio (omega) of the number of nonsynonymous substitutions per non-synonymous site (\textit{dN}), to the number of synonymous substitutions per synonymous site (\textit{dS}). If EVEs are evolving neutrally, then the ratio is expected to be equal to 1, whereas if the EVE is under purifying selection, \textit{dN/dS} is expected to be lower than 1. We conducted this analysis on trimmed codon alignments from (\figurename{ \ref{figure:Bioinformatic_pipeline}}-11) via the codeml algorithm from PAML \citep{yang_paml_2007} used through the ETE3 package \citep{huerta-cepas_ete_2016}  (model Muse and Gaut  \citep{muse_likelihood_1994}). We then used a branch model to test the deviation from the null model in which marked branches (called foreground) wich corresponded to the monophyletic EVE clade evolved under a neutral scenario ($\chi^2$ test). The \textit{dN/dS} estimated for the whole clade is then the average of each branch of the clade. The p-values were then adjusted by selecting an FDR of 0.05 \citep{puolivali_influence_2020}, and we estimated the standard errors of \textit{dN/dS} that maximized the likelihood (option getSE = 1). \textit{dN/dS} with dS greater than 10 were removed, since this indicates substitution saturation (\figurename{ \ref{figure:Bioinformatic_pipeline}}-12). 

The other way we choose to study the domesticated nature of a viral gene was to study their expression profile (\figurename{ \ref{figure:Bioinformatic_pipeline}}-13). We reasoned that domesticated genes are likely to be significantly expressed.  To test this, when RNAseq reads were available on NCBI (SRA), we mapped them on the assembled genomes (until reaching 300x coverage as far as possible). Using the TPMCalculator program \citep{vera_alvarez_tpmcalculator_2019}, we measured expression in ovaries and whole body if available or alternatively in any tissue (see supplemental information table :RNA\_seq\_reads\_mapped.txt). An EVE was considered as domesticated if the gene was expressed with a Transcripts Per Kilobase Million (TPM) index above 1000. This threshold was chosen based on the median value observed for control EVEs (718.70 TPM), rounded up to 1000TPM to be conservative. We measured the accuracy of this metric using EVEs for which both TPM and \textit{dN/dS} calculations were possible: among the 36 genes having a TPM$>$1000, 33 also had a \textit{dN/dS} significantly below 1 suggesting that inferring domestication based on TPM$>$1000 was consistent with \textit{dN/dS} test with a  0.9166 probability.

Finally, based on the idea that an active EVE should encode a protein with similar length to the donor virus, we calculated the actual viral protein sequence length using the orfipy algorithm \citep{singh_orfipy_2021} (\figurename{ \ref{figure:Bioinformatic_pipeline}}-14).  

A possible bias when comparing the effect of lifestyles on domesticated elements could come from a difference of RNAseq reads availability depending on the lifestyle, which may result in a different number of EVEs considered as domesticated. A GLM binomial analysis did not reveal any correlation between RNAseq data availability and lifestyle (endoparasitoid = Slope(SE)=0.21(0.62), p=0.73; free-living= Slope(SE)=0.40(0.57), p=0.49 using ectoparasitoid as intercept).

\subsection{Sensitivity and specificity of the analysis}
\label{sec:MM-8}

\textbf{\\Capacity to find Endogenous Viral Elements (EVEs)}
\\Among the species included in our dataset, 7 were known to contain a domesticated virus (2 with similar PDV \citep{bezier_polydnaviruses_2009-1}, 5 with different VLPs \citep{pichon_recurrent_2015,burke_common_2019,di_giovanni_behavior-manipulating_2020}), corresponding to 4 independent endogenization events.  Our pipeline was able to detect the vast majority of the corresponding virally-derived genes (88.6\%, details in table \ref{tab:Control_table}). The 11.14\% false negatives corresponded to sequences that were too divergent or with a region of similarity too small to be detected by our pipeline. We found that 88.7\% of the control EVEs were located within scaffolds scored as A (i.e. having a depth of coverage falling within the distribution of those containing BUSCO genes, as well as having one or more eukaryotic genes and/or transposable elements in the vicinity). Since the remaining 11.3\% were scored either C (7.64\%) or D (3.66\%) (table \ref{tab:Control_table}), we considered candidates within the range A-D as valid candidates for endogenization. On the contrary, scaffolds annotated as F or X were rather considered as free-living viruses since they did not show eukaryotic genes or TE in their vicinity and had different coverage compared to BUSCO-containing scaffolds. Scaffolds classified as E were of unclear status and discarded.

\textbf{\\Capacity to find domesticated EVEs (dEVEs)}
\\An EVE was considered as domesticated if the \textit{dN/dS} ratio was significantly below 1 or if TPM was above 1000. When \textit{dN/dS} computations were possible (for 75/152 control EVEs), our pipeline considered the control EVEs as being under purifying selection in 70.39\% of the cases. Overall, by combining the two metrics (\textit{dN/dS} and TPM), our pipeline identified 69.04\% of the control locus as being domesticated (table \ref{tab:Control_table}).

\textbf{\\Capacity to infer events of endogenization (EVEs events)}
Among the control species, the pipeline correctly inferred the expected 4 independent events: (1) \textit{Leptopilina boulardi}/\textit{Leptopilina clavipes}/\textit{Leptopilina heterotoma} (Di Giovanni et al., 2020) (2) \textit{Venturia canescens} \citep{pichon_recurrent_2015}, (3) \textit{Fopius arisanus} \citep{burke_rapid_2018}, and (4) \textit{Cotesia vestalis}/\textit{Microplitis demolitor} \citep{bezier_polydnaviruses_2009} (table \ref{tab:Control_table}).
However, in addition to the expected unique shared event concerning the \textit{M. demolitor} and \textit{C. vestalis} species, our pipeline inferred two additional events, each specific to one lineage.  This was due to the fact that two genes were not detected by our pipeline as shared by \textit{M. demolitor} and \textit{C. vestalis}, either because they are effectively not shared (for 3 of them: HzNVorf118, like-\textit{pif-4} (\textit{19kda}), \textit{fen-1}), or because of some false negative in one of the two lineage (for one  of them:\textit{p33} (\textit{ac92})).


\subsection{Assessing  the distribution of virus infecting insects}
\label{sec:MM-8}
We estimated the number of viral species infecting insect species based on the virushostdb database (downloaded the 23/05/2022 on https://www.genome.jp/virushostdb/) which lists a wide diversity of viral species associated with their putative hosts. We also added two important exploratory RNA virus search studies \citep{shi_redefining_2016, wu_abundant_2020}. We kept only viruses found in interaction with insect in at least one of these databases. Genomic structures were retrieved through the ICTV report (2021.v1) and information available in ViralZone (all viral species details can be found on the GitHub file: \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/All_virus_infecting_insects_informations.csv}{All\_virus\_infecting\_insects\_informations.csv}). We counted the number of viruses per genomic structure, and viruses from unknown genomic structures were discarded.  In total, 2,626 viral species infecting insects were considered (detail : ssRNA(-) = 603sp, ssRNA(+) = 1,241sp, ssDNA = 75sp, dsRNA = 401sp, dsDNA =155sp, Unknown= 151sp). The Partiti-Picobirna, Narna-Levi, Mono-Chu, Bunya-Arenao, Luteo-Sobemo, Hepe-Virga and Picorna-Calici clades correspond to viral clades proposed by \citep{shi_redefining_2016}.

\subsection{Divergence time estimation}
\label{sec:MM-10}
We time-calibrated the inferred phylogenetic tree using a Bayesian approach on RevBayes 1.1.1 \citep{hohna_probabilistic_2014} and information on 5 fossils selected by \citep{peters_evolutionary_2017}. Reduction of the supermatrix became necessary to overcome computational limitations when estimating node ages resulting from the large size of the concatenated BUSCO supermatrix (nsites = 228,009). We then generated one fasta file with a random draw without replacement of 20,000 sites from the supermatrix.
Evaluation of the phylogenetic likelihood being the most expensive operation when calculating the posterior density, we decided to use the method developed in \citep{szollosi_relative_2020} to reduce computational cost and approximate the phylogenetic likelihood using a two-step approach. In the first step, the posterior distribution of branch lengths measured in expected number of substitutions is obtained for the fixed unrooted topology of using a standard MCMC analysis (100,000 iterations, 3 chains, 5000 burn-in, tuningInterval=200). The obtained posterior distribution is then used to calculate the posterior mean and posterior variance of the branch length for each branch of the unrooted topology. In the second step, we date the phylogeny using a relaxed clock model and calibrations (500,000 iterations, 4 chains, 5000 burn-in, tuningInterval=200). Calibration of the root was done using a uniform law between 300 and 412 Mya. To verify that MCMC analyses converged to the same posterior distribution,
for both steps we computed the effective sample size and applied the Kolmogorov-Smirnov test using the package convenience v1.0.0 with a minimum ESS threshold of 100 (however, due to an excessive demand for resources, we were unable to achieve the sampling value of an ESS of minimum 100 for 46/389 parameters (min=44.25)).

\subsection{Ancestral state reconstruction}
\label{sec:MM-11}
To explore the dynamics of EVEs gain in relation to lifestyle, we first had to reconstruct the ancestral lifestyle states of the Hymenoptera used in this study. This was achieved using a Bayesian approach implemented in RevBayes 1.1.1 \citep{hohna_probabilistic_2014}. The lifestyles of the Hymenoptera species used in this study were deduced from various sources (details and sources in the GitHub table named : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/Assembly_genome_informations.csv}{Assembly\_genome\_informations.csv}). Since lifestyle characters are probably not equally likely to change from any one state to any other state, we choose the Mk model with relaxed settings allowing unequal transition rates. Thus, we assumed 6 different rates with an exponential prior distribution. Before running the MCMC chains, we made a preliminary MCMC simulation used to auto-tune the moves to improve mixing of the MCMC analysis with 1000 generations and a tuning interval of 300. We then ran two independent MCMC analyses, each set to run for 200 000 cycles, sampling every 200 cycles, and discarding the first 50 000 cycles as burn-in. To verify that MCMC analyses converged to the same posterior distribution, we computed the effective sample size and applied the KolmogoRov-Smirnov test using the package convenience v1.0.0 with a minimum ESS threshold of 100.  The MCMC chain was subsampled to provide 1000 samples. At each sample, ancestral states were reconstructed for all nodes of the phylogeny. We assumed that the state assigned to a node was constant throughout the branch leading to that node.

\subsection{Test of the lifestyle effect on viral endogenization and domestication}
\label{sec:MM-12}

In order to test the lifestyle effect on the propensity to integrate and domesticate viral element, we first randomly sampled 1000 probable ancestral state scenarios to take into account the uncertainty in the estimates of the ancestral states of the nodes. Because a lot of branches had no EVE endogenization inferred, we ran zero-inflated negative-binomial GLM model, for each of these 1000 scenarios such that (GLM(NbEVEs\char`\~ free-living + endoparasitoid + ectoparasitoid * Branch\_length, family = zero inflated neg binomial). We eliminated all branches older than 160 million years because they are too old for our analysis to detect events (the oldest event detected by our analysis is around 140 mya) that could artificially inflate the zero count. The model was implemented in stan language using the R package brms (seed = 12345, thin=5, nchains =4, niter = 10000) \citep{burkner_brms_2017,burkner_advanced_2018}. Posterior predictive check was done using the package brmsfit in order to check that the model was correctly predicting the proportion of zeros.  Indices relevant to describe and characterize the posterior distributions were computed using the R package BayestestR \citep{makowski_bayestestr_2019}. Autocorrelation was studied using the effective sample size index (ESS) with a value greater than 1000 being sufficient for stable estimates \citep{burkner_brms_2017}. The convergence of Markov chains was evaluated by a Rhat statistic equal to 1. All the posterior coefficient estimated values were then pooled together (after checking the convergence of all chains via the GelmanRubin function in R \citep{bolstad_understanding_2010}) and compared between the free-living, endoparasitoid and ectoparasitoid modalities.  

To calculate the rate of domestication independent of the rate of endogenization, we built a binomial logistic regression model in a Bayesian framework, specifying the number of domesticated EVEs (or Events) (the numerator) relative to the total number of EVEs or Events inferred by our pipeline (the denominator). These binomial models allowed us to test whether the probability of domestication after endogenization correlated with lifestyle by controlling for the endogenization input (the denominator). Thus, for each of the 1000 lifestyle scenarios, we ran a binomial brms model with a logit link such that brms(Nb dEVEs/dEvents | trials(Nb EVEs/Events) \char`\~ lifestyle + Branch length).

Before analyzing the data, we checked that the inferences did not depend on the quality of the genomes selected for analysis. We found a significant effect of the lifecycle on the N50 and percentage of complete+partial BUSCO in the assemblies (Kruskal-Wallis rank sum test p-values respectively = 3.192e-10 and 1.26e-14). Furthermore, a pairwise Wilcox test with p-values adjusted with Bonferroni method revealed a significantly higher values of N50 and \%complete+partial BUSCO in genome assemblies from free-living species compared to endo and ectoparasitoids species (p-value $<$ 0.05). The same test using the total assembly length in bp did not reveal any difference between the three lifestyles (p-value $>$0.05). Overall, free-living species have better assemblies. Because better assembly quality should facilitate the discovery of endogenous viral elements both by sequence homology detection and by a better assessment of the endogenized nature of the EVE (scaffolds A,B,C and D), we should thus underestimate the number of EVEs in endo- and ectoparasitoid species compared to free-living species. Since our analysis led to opposite conclusion, our results cannot be explained by this feature of the dataset.

\section{Supplementary informations}

\subsection{ssRNA endogenization}
\label{sec:SI-1}

Although our results show an under-representation of EVEs deriving from ssRNA viruses (relatively to their high abundance in insect virome), they were involved in a high absolute number of endogenization events: 21 viral families/clades were involved in 174 independent endogenization events in the 114 Hymenoptera genomes analyzed, in particular involving \textit{Chuviridae}, \textit{Artoviridae} and \textit{Nyamiviridae} (\figurename{\ref{figure:Genomic_structure_and_families_Events_distributions}-B}). In a recent meta-analysis \citep{gilbert_diversity_2022}, more than 1876 EVEs involving ssRNA viruses were identified in 37 species distributed in 8 insect orders. Interestingly, the authors noticed that the contribution of negative-stranded RNA viruses was overall high (67\%), but was also highly species-specific. In our dataset, the great majority of ssRNA viruses donors were negative-stranded (14/21 viral family/clades) accounting for 78.8\% of ssRNA events. The pattern thus seems even more pronounced in Hymenoptera compared to insects in general, and resemble the pattern observed in ticks \citep{russo_novel_2019}. The reasons for the asymmetry observed between negative and positive strand RNA viruses in endogenization are unclear. One explanation proposed by \citep{holmes_evolution_2011} posits that the ssRNA(-) have a higher probability of endogenization compared to ssRNA(+) because non-segmented ssRNA(-) usually produce abundant short mRNAs compared to ssRNA(+) which conversely produce lower amount of long mRNAs encoding a single polyprotein \citep{modrow_viruses_2013}. Then, all else being equal, an RNA(-) virus would produce more RNA molecules, which will increase the likelihood that some of them get reverse-transcribed and ultimately endogenized into the host genome. In support of that view, \citep{holmes_evolution_2011} noticed that the NP gene was more often endogenized compared to the other genes encoded by most ssRNA(-), which fits its prediction. This is because  for most Mononegavirales species the 3' nucleoprotein (NP) gene is the most abundant RNA \citep{compans_transcription_2004}, due to the polar 3'-5' stepwise attenuation of transcription \citep{compans_transcription_2004}. This pattern was since observed on some systems (i.e. mosquitoes and few mammals genomes \citep{katzourakis_endogenous_2010}) but opposite results were also obtained on others (ticks, \citep{russo_novel_2019}). In our dataset, we found two ssRNA(-) non-segmented families showing the expected pattern where the genes closest to the 3' regions were the most endogenized : the Nucleoprotein (N) which is the first transcribed protein in \textit{Nyamiviridae} was endogenized in 26 cases out of 28, and the 3'-unknown protein in \textit{Lispiviridae} (first transcribed) was endogenized in 12 cases out of 15 endogenization events (\tablename{\ref{tab:single-stranded_RNA_orf_location}}). On the contrary, all the other putative  ssRNA families donor presented more EVEs deriving from the middle or the 5' genomic regions: the most endogenized gene from \textit{\textit{Artoviridae}} was the U2 protein (19/39) which is in the middle of the genome (2nd/3); in \textit{Bornaviridae} and \textit{Rhabdoviridae} the most endogenized gene was the RdRp (L) protein, which is the last ORF in the first genome (out of 8 genes) and the one just before the last gene in \textit{Rhabdoviridae}. Finally, out of 36 EVEs deriving from \textit{Chuviridae}, 26 corresponded to the Nucleoprotein (N) which is the last transcribed protein in the closest viral genomes. These unexpected results under Holmes model may thus lead one to reject the hypothesis, unless peculiar mechanisms of regulation of the transcription are at play for these viruses. Another explanation could come from a strong selective pressure for retaining particular proteins (i.e. Nucleoprotein) in the genome, independently of their level of transcription.

\subsection{A new case of virus domestication in \textit{Platygaster orseoliae}}
\label{sec:SI-2}

In the assembly of \textiti{P. orseoliae}, 12 scaffolds were annotated as free-living viruses (F-X scaffolds). They had a different sequencing depth compared to BUSCO containing scaffolds and encoded 136 complete ORFs for which 21 presented homology with LbFV ORFs (min bit score = 50,  min ORF size = 150pb, max overlaps = 23pb). ORF density was 82.7\% which is in the range of expected values for related free-living viruses \citep{bezier_genome_2015}. In order to identify additional scaffolds possibly belonging to this free-living virus, we searched for homology between the 136 putative viral proteins, and the scaffolds obtained from the assembly of \textit{P. orseoliae}. These results allowed us to identify two additional scaffolds (scaffold\_117128 and scaffold\_18896). Because the total size of the 14 putative "free-living" scaffolds was within the expected range for a dsDNA virus genome (136,801 bp) and because the average coverage was much higher than BUSCO-containing scaffolds (mean cov = 95.6X [sd=5.05X] compared to 33X in BUSCOs) and homogeneous (\figurename{ \ref{figure:Multiple_EVEs_COV_GC}}), we believe that this set of scaffolds corresponds to the whole genome of a new virus, related to LbFV, which we propose to call Platygaster orseoliae filamentous virus (PoFV). In order to characterize the relationship of this new virus within dsDNA viruses diversity, we inferred a phylogenetic tree including ORFs of known dsDNA viruses along with the EVEs newly identified here. The phylogenetic reconstruction revealed that PoFV was the closest relative of the endogenized virus found in the same species (PoEFV, \figurename{ \ref{figure:dsDNA_phylogeny}}). 

In order to detect possible new viral endogenization from the same "donor virus", we queried the genome of \textit{P. orseoliae} with the 136 predicted proteins of PoFV. This way, we found a total of 139 convincing hits (89 PoFV ORFs), including the hits to the 15 ORFs with LbFV-homology. All ORFs were encoded by scaffolds with BUSCO-like coverage depth (p-value cov $>$= 0.05) and/or containing eukaryotic genes and/or transposable elements (Blastx E-value max = 7.060e-07, bits min =50, with an average percentage of identity of 69.16\%). Furthermore, a large proportion of the EVEs (22.7\%) presented premature stop codons within the sequences, further suggesting that these virally-derived genes are indeed endogenized since abundant pseudogenization is not expected in free-living virus genomes (\figurename{ \ref{figure:Porseoliae_endogenization_events_gene_plot}}-A).

Among the 81/139 apparently intact EVEs (with ORF length $>$= 50\% of the PoFV ORF), some are likely implicated in DNA replication (\textit{integrase}), in transcription (\textit{lef-8},\textit{lef-9},\textit{lef-5},\textit{lef-4}), in packaging and envelopment (\textit{ac81}, \textit{38k}) and in infectivity (\textit{pif-1}, \textit{pif-2}, \textit{pif-3}). Among the 139 PoFV-related EVEs found in \textit{P. orseoliae}, 104 corresponded to putative paralogs. Conversely, none of these 104 ORFs were present in two copies within the PoFV genome, suggesting that a major post-endogenization duplication event occurred or that multiple endogenization events did occur. Among these 104 duplicated EVEs, 78 presented topologies allowing us to calculate \textit{dN/dS} ratios using Bayesian pairwise estimates (runmode -3 in codeml) or foreground/background tests (codeml) when topologies presented more than 4 leafs. Before running the foreground/background tests, we constrained all paralogs to form a monophyletic group including the PoFV loci as the closest taxa in the phylogenies (all LRT tests did not significantly present differences between constrained and unconstrained trees). 
Among these 78 paralogs EVEs, 44 presented a complete and intact open reading frame and a \textit{dN/dS} ratio significantly lower than 1 suggesting that they are under stabilizing selection (\figurename{ \ref{figure:Porseoliae_endogenization_events_gene_plot}}-A). \\

Although functional studies are needed to confirm that these virus-derived genes are involved in the formation of VLPs as observed in \textit{Leptopilina} (Di Giovanni et al., 2020), we believe that \textit{P. orseoliae} filamentous virus (PoEFV) is a good candidate for viral domestication, possibly involved in counteracting its Diptera host immune system (from the family Cecidomyiidae \citep{buhl_new_2016}).\\

\subsection{Assignation of the unknown Hymenoptera to species}
\label{sec:SI-2}

UCEs along with 400 bp of flanking regions on either side were extracted from the low coverage scaffolds with a custom script. We used a two-step process to assign the unknown sample to species. First, UCEs + flanking regions were analyzed with a set of UCEs + flanking regions obtained from early diverging families of Chalcidoidea by \citep{cruaud_optimized_2019, rasplus_first_2020} to assign the unknown sample to family. Then, unknown sequences were analyzed with a larger set of species belonging to the identified family (Eulophidae; loci taken from \cite{rasplus_first_2020}). In both cases, only loci that had a sequence for at least 75\% of the samples included in the analysis were retained. Loci were aligned with MAFFT (-linsi option; \citep{katoh_mafft_2013}). Positions with $>$ 90\% gaps and sequences with $>$ 25\% gaps were removed from the alignments using SEQTOOLS (package PASTA; \citep{mirarab_pasta_2015}). The concatenation of all loci was analysed with IQ-TREE v 2.0.6 \citep{minh_iq-tree_2020} without partitioning. Best fit models were selected with the Bayesian Information Criterion (BIC) as implemented in ModelFinder (\citep{kalyaanamoorthy_modelfinder_2017}). FreeRate models with up to ten categories of rates were included in tests. The candidate tree set for all tree searches was composed of 98 parsimony trees + 1 BIONJ tree and only the 20 best initial trees were retained for NNI search. Statistical support of nodes was assessed with ultrafast bootstrap (UFBoot2) (\citep{hoang_ufboot2_2018}) with a minimum correlation coefficient set to 0.99 and 1,000 replicates of SH-aLRT tests (\citep{guindon_new_2010}). Results of the phylogenetic analyses are presented in \figurename{\ref{figure:UCEs}}. Placement of the unknown species in trees shows that samples of \textit{P. orseoliae} were likely mixed up with a species belonging to the genus \textit{Aprostocetus} (Eulophidae, Tetrastichinae). Given its small size, color and abundance (265 species described just in Europe), it seems plausible that one specimen of \textit{Aprostocetus} sp. remained unnoticed in the pool of \textit{P. orseoliae}.


% \usepackage{array}
% \usepackage{graphicx}
% \usepackage{booktabs}

% \usepackage{array}
% \usepackage{graphicx}
% \usepackage{booktabs}

% \usepackage{array}
% \usepackage{graphicx}
% \usepackage{booktabs}


\section{Supplementary figures}

\begin{figure}[H]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Genomes_qualities_and_origins.pdf}
\caption[Paper1:Genomic dataset sources]{\textbf{Source of the datasets and availability of the reads}. Phylogeny of 124 Hymenoptera species. Two Coleoptera species were used to root the tree. The aLRT bootstrap scores are represented along the nodes. The sources refer to the platform or laboratory in which the assemblages come from (This study, BIPPA: BioInformatics Platform for Agroecosystem Arthropods, NCBI: National Center for Biotechnology Information). The assemblies for which raw DNAseq or RNAseqs reads were available are listed in the column DNA or RNA reads. The G+C\% column reflects the average G+C rate for each assembly, and the BUSCO\% column reflects the rate of complete or partial BUSCOs found via the analysis with BUSCO V3. Posterior Bayesian lifestyle inference distribution for each node and tips  are represented by colored pie charts. }
\label{figure:Genomes_qualities_and_origins}
\end{figure}

\begin{figure}[H]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/UCEs.pdf}
\caption[Paper1:UCE phylogenies to describe the unknown scaffolds]{\textbf{UCE trees built to assign to species the unknown Chalcidoidea sequenced with the pool of \textit{P. orseoliae}}. \textbf{A}: Phylogeny of early diverging families of Chalcidoidea (423 UCES and 127,979 bp were analysed to get the tree, best fit model = GTR+F+R10). \textbf{B} : Phylogeny of the family Eulophidae to which the unknown sample was inferred to belong to (408 UCES and 77,514 bp were analysed to get the tree, best fit model = GTR+F+R10). For both trees, SH-aLRT/UFBoot are shown at nodes; the number of UCEs analyzed for each sample is indicated between bracket and the unknown sample is highlighted in red.}
\label{figure:UCEs}
\end{figure}


\begin{table}[H]
\centering
\setlength{\extrarowheight}{0pt}
\addtolength{\extrarowheight}{\aboverulesep}
\addtolength{\extrarowheight}{\belowrulesep}
\setlength{\aboverulesep}{0pt}
\setlength{\belowrulesep}{0pt}
\caption[Paper1:Control table]{\textbf{Summary statistics for control cases}. The numerator indicates the numbers of EVEs or dEVEs inferred by our pipeline, and the denominator indicates the number of known EVEs for each case.  Analysis on \textit{dN/dS} was only possible when orthologs or paralogs were available.  Controls Endogenous viral elements present in scaffolds probably belonging to the Hymenoptera genome are scored from A to D, and scaffolds probably belonging to free viruses are scored as F or X: (see details in \hyperref[sec:MM-5]{Materials and methods}). TPM (Transcripts per kilobase million) values were calculated via RNAseq read mapping when available in the databases (all RNAseq data sources can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/RNA_seq_reads_mapped.txt}{RNA\_seq\_reads\_mapped.txt}). * In addition to the expected unique shared event concerning the \textit{M. demolitor} and \textit{C. vestalis} species, our pipeline inferred two additional events, each specific to one lineage. This was due to the fact that two genes were not detected by our pipeline as shared by \textit{M. demolitor} and \textit{C. vestalis}, either because they are effectively not shared (for 3 of them: HzNVorf118, like-\textit{pif-4} (\textit{19kda}), \textit{fen-1}), or because of some false negative in one of the two lineage (for one of them:p33 (ac92)). }
\resizebox{\linewidth}{!}{%
\begin{tabular}{>{\hspace{0pt}}m{0.13\linewidth}|>{\hspace{0pt}}m{0.109\linewidth}|>{\hspace{0pt}}m{0.091\linewidth}|>{\hspace{0pt}}m{0.089\linewidth}>{\hspace{0pt}}m{0.106\linewidth}|>{\hspace{0pt}}m{0.089\linewidth}>{\hspace{0pt}}m{0.113\linewidth}>{\hspace{0pt}}m{0.089\linewidth}|>{\hspace{0pt}}m{0.08\linewidth}} 
\toprule
 & \textit{V. canescens} & \textit{F. arisanus} & \textit{C. vestalis} & \textit{M. demolitor} & \textit{L. boulardi} & \textit{L. heterotoma} & \textit{L. clavipes} & \% Total \\ 
\hhline{--------~}
{\cellcolor[rgb]{0.914,0.918,0.925}}\textbf{NB EVEs\footnotemark} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{36/40~} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{42/47~} & {\cellcolor[rgb]{0.906,0.91,0.918}}\textbf{20/21} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{18/25~} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{12/13} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{12/13} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{12/13~} & \textbf{88.4\%} \\
Scaffold A & 36  & 42  & 18 & 16 & 11 & 6 & 8  & 90.13\% \\
Scaffold B & 0 & 0 & 0 & 0 & 0~ & 0~ & 0 & 0\% \\
Scaffold C & 0 & 0 & 0 & 0 & 1 & 6 & 4  & 7.24\%  \\
Scaffold\textasciitilde{} D & 0 & 0 & 2 & 2 & 0 & 0 & 0 & 2.63\% \\
Scaffolds E-F-X & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\% \\ 
\hhline{--------~}
{\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{NB dEVEs} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{22/36~} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{11/42} & {\cellcolor[rgb]{0.922,0.914,0.925}}\textbf{20/20} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{18/18} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{12/12} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{12/12} & {\cellcolor[rgb]{0.914,0.91,0.918}}\textbf{12/12~} & \textbf{71.82\%} \\
by \textit{dN/dS}\footnotemark & 3/3 & 0/1 & 16/20 & 15/18  & 12/12 & 12/12 & 12/12  & 70.39\% \\
by TPM   1000 & 21 & 11 & 10 & 9  & 1 & 0 & 0 & 43.60\% \\
\hhline{--------~}
{\cellcolor[rgb]{0.898,0.894,0.902}}\textbf{Nb Events} & {\cellcolor[rgb]{0.898,0.894,0.902}}\textbf{1/1} & {\cellcolor[rgb]{0.898,0.894,0.902}}\textbf{1/1} & \multicolumn{2}{>{\centering\hspace{0pt}}m{0.195\linewidth}|}{{\cellcolor[rgb]{0.937,0.933,0.941}}\textbf{3/1*}} & \multicolumn{3}{>{\centering\hspace{0pt}}m{0.291\linewidth}|}{{\cellcolor[rgb]{0.929,0.922,0.933}}\textbf{1/1}} & . \\
\bottomrule
\end{tabular}}
%\raggedright{1 The total percentage is the average of the averages of the percentages found in each of the 4 events \footnotetext[1]{}}
\\\raggedright{1 When paralogs were detected on different scaffolds, the best scaffold score was used.\footnotetext[1]{}}
\\2 Analysis on \textit{dN/dS}was only possible when orthologs or paralogs were detected (for example, in \textit{V. canescens} this calculation was only possible for 3 genes having paralogs)\footnotetext[2]{}
\label{tab:Control_table}
\end{table}


% \usepackage{array}
% \usepackage{graphicx}
% \usepackage{booktabs}
% \usepackage{colortbl}


\begin{table}
\centering
\captionsetup{font=footnote}
\caption[Paper1:Summary table with all EVEs statistics]{
\textbf{Summary table}. Clusters refers to the number of homologous clusters with one or more candidate Endogenous Viral Elements (EVEs). Raw EVEs is the raw number of EVEs (i.e. including all paralogs and orthologs) according to the scaffold categories from A to D.
Nb EVEs with complete ORFs corresponds to the number of raw EVEs with an ORF starting with a methionine, without premature stop codons, and ending with a stop codon, distinguishing ORFs whose size is at least equal to or greater than 80\% of that of the best viral hit. 
EVEs is the count of EVEs, i.e. counting the number of genes within each monophyletic group only once (as several EVEs may have undergone post-endogenization duplications or an EVE may be ancestrally acquired and shared by several species). Mean pident is the average of the percentage identity of all EVEs with the best viral hit, the number in brackets corresponds to the standard error. Nb EVEs pident and Nb EVE E-values 1e-20 correspond to the number of refined EVEs showing a hit with more than 80\% identity and an E-value below  1e-20 with a viral protein, respectively. 
Domesticated EVEs (dEVEs) corresponds to the number of refined EVEs with either a \textit{dN/dS} significantly less than 1 with a complete ORF and without a stop codon and/or a TPM value $>$ 1000 with a complete ORF and without stop codon.
Events corresponds to the number of endogenization events that may include one or more genes and involve one or more species. Shared Events is the number of endogenization events shared by at least two species. Viral families corresponds to the number of different putative viral families associated with the best viral hits.  dEvents corresponds to the number of endogenization events presenting at least one dEVE.}
\setlength{\extrarowheight}{0pt}
\addtolength{\extrarowheight}{\aboverulesep}
\addtolength{\extrarowheight}{\belowrulesep}
\setlength{\aboverulesep}{0pt}
\setlength{\belowrulesep}{0pt}
\resizebox{\linewidth}{!}{%
\begin{tabular}{>{\hspace{0pt}}m{0.223\linewidth}>{\hspace{0pt}}m{0.174\linewidth}>{\hspace{0pt}}m{0.129\linewidth}>{\hspace{0pt}}m{0.131\linewidth}>{\hspace{0pt}}m{0.14\linewidth}>{\hspace{0pt}}m{0.107\linewidth}} 
\toprule
 & dsDNA & ssDNA & dsRNA & ssRNA & Total \\ 
\hline
Clusters & 166 & 9 & 18 & 36 & 229 \\ 
\hline
Raw EVEs \par{}(A\textbar{}B\textbar{}C\textbar{}D) & 819\par{}(534\textbar{}61\textbar{}102\textbar{}122) & 92\par{}(40\textbar{}27\textbar{}6\textbar{}19) & 95\par{}(45\textbar{}19\textbar{}26\textbar{}5) & 255\par{}(129\textbar{}74\textbar{}34\textbar{}18) & 1261 \\
Complet raw EVEs ORFs\par{}($>$80\% best viral hit) & 694 (409) & 74 (39) & 76 (26) & 223 (76) & 1067 (550) \\
\rowcolor[rgb]{0.91,0.91,0.91} EVEs & 366 & 32 & 61 & 162 & 621 \\
Mean pident (sd) & 36.99 (15.35) & 36.91 (11.37) & 38.32 (13.27) & 33.30 (11.82) & 36.32 \\
EVE pidents$>$>= 80 & 22 & 0 & 0 & 0 & 22 \\
EVE E-values  1e-20 & 430 & 29 & 67 & 126 & 652 \\
dEVEs & 112 & 9 & 12 & 38 & 171 \\
\rowcolor[rgb]{0.91,0.906,0.91} Events & 130 & 26 & 59 & 152 & 367 \\
shared Events & 17 & 1 & 5 & 13 & 36 \\
viral families/clades & 12 & 3 & 4 & 21 & 40 \\
dEvents & 47 & 10 & 12 & 38 & 107 \\
\bottomrule
\end{tabular}
}
\label{tab:Summary_candidate_numbers}
\end{table}


\begin{figure}[!htpb]
\captionsetup{font=footnotesize}
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Cluster21304_AA.dna.treefile.pdf}
\caption[Paper1:Canonical example of an endogeniation events]{\textbf{Example of endogenization events}. The phylogeny of cluster21304 corresponds to the clustering of a set of viral and candidate viral insertion genes sharing a homology. In red are represented the loci of viral origin, and in blue are represented the loci probably endogenized (EVEs). The letter at the end of the taxon label represents the endogenization score assigned to the candidate (see details in \hyperref[sec:MM-5]{Materials and methods}). In this example, we found two singular endogenization events in the species endoparasitoid \textit{Encarsia formosa} (annotated A and thus presenting a depth of coverage non-significantly different from the distribution of the BUSOs of the genome as well as at least one transposable element and/or one eukaryotic gene) and ectoparasitoid \textit{Torymus auratus} (annotated C and thus presenting only a depth of coverage non-significantly different from the distribution of the BUSOs of the genome). Since these two species do not share a close common ancestor in the phylogeny and come from two different families, the algorithm therefore assigned them to two independent viral endogenization events. The viral locus found in the assembly of the endoparasitoid species \textit{Platygaster orseoliae} was annotated F, meaning that the depth of coverage deviated significantly from the BUSCO distribution of the genome and that no TEs and less than 5 eukaryotic genes were found in the scaffold containing the candidate insertion. Finally, the two loci belonging to the ectoparasitoid species \textit{Megastigmus stigmantizans} and \textit{Megastigmus dorsalis} both show a score supporting viral endogenization. Furthermore, these species exhibit a doubly monophyletic clade (high bootstrap score) within the gene phylogeny and within the species phylogeny, suggesting that they acquired this viral gene from their closest common ancestor about 20 million years ago. All newick phylogenies are available on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/All_cluster_phylogenies_merged.pdf}{All_cluster_phylogenies_merged.pdf}.}
\label{figure:Cluster21304_phylogeny_example}
\end{figure}


\begin{figure}[!htpb]
\captionsetup{font=footnotesize}
 \centering
    \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Multiple_EVEs_COV_GC.pdf}
\caption[Paper1:GC and coverage distribution of scaffolds containing multiple EVEs Events]{\textbf{G+C\%Coverage distribution of scaffold containing multiple EVEs Events}.  The size of the dots corresponds to the number of candidate EVEs inside the scaffold. The color represents the genomic entity from which the EVE probably originated (brown: \textit{Nudiviridae}, blue = LbFV and green = AmFv). The red color refers to scaffolds showing free-living virus signatures. The grey color refers to scaffolds containing one or more BUSCO genes. The dots circled in black correspond to scaffolds that contain one or more eukaryotic genes and/or repeat elements.}
\label{figure:Multiple_EVEs_COV_GC}
\end{figure}


\begin{figure}[!htbp]
\captionsetup{font=footnotesize}
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/All_violin_plots_new_order2.pdf}
\caption[Paper1:Violon plot distribution of GLM coefficients of EVEs,dEVEs,Events and dEvents for each genomic viral structure]{\textbf{Violin plots of the posterior distribution of GLM coefficients after exponential transformation in relation to wasp lifestyle}. The ectoparasitoid lifestyle is in yellow, the endoparasitoid lifestyle is in green, and the free-living lifestyle is in blue. Coefficients have been transformed into exponential and correspond to the posterior distribution of the coefficients of a binomial negative zero-inflated GLM model, where the lifestyle free-living stand for the intercept. The Y-axis corresponds to the multiplicative factor of the number of endogenization and/or domestication of EVES and/or events relative to free-living species. The coefficients are derived from 1000 GLM models adjusted on 1000 randomly selected probable scenarios ($>$90 CI) of ancestral states at nodes. Branches from nodes older than 160 million years have been removed from the dataset. The ROPE\% is the percentage of the posterior distribution of coefficients below the intercept. The posterior distribution of the interaction coefficients between lifestyles and branch size were not informative, and the branch size factor was therefore added as an additive effect to the model.}
\label{figure:All_violin_plots}
\end{figure}


\begin{figure}[!htpbt]
\includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Porseoliae_LbFV-like_phylogenies.pdf}\centering
\caption[Paper1:\textit{P. orseoliae} integrase and Ac81 EVE phylogenies]{\textbf{Phylogenies of LbFV-like proteins under purifying selection in \textit{Platygaster orseoliae} genome}. The panel A represents the Cluster\_25710 which corresponds to the \textit{integrase} protein. The panel B represents the Cluster\_26675 which corresponds to the \textit{Ac81} protein. Taxa in red correspond to putative  loci belonging to free-living viruses, while green taxa correspond to putative EVEs (the assigned family is indicated after the pipe). Green labels indicate the following information : (1) scaffold name, (2) start location, (3) end location, (4) strand, (5) Hymenoptera species, (6) endogenization index, (7) the inferred event number within the cluster phylogeny, and (8) whether the loci are found under purifying selection (YES) or not (NO).}
\label{figure:Porseoliae_LbFV-like_phylogenies}
\end{figure}

%\begin{figure}[!htpbt]
%\includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{GO_distribution_dEVEs_vs_freeliving_dsDNA_viruses.pdf}\centering
%\label{figure:EVE_dEVEs_sementic_spaces_comparison}
%\caption{\textbf{Semantic space distribution of typical dsDNA viral Go terms compared to endoparasitoid candidate EVEs Go terms}.  The GO terms semantic space distribution of dsDNA virus is based on the proteomes of 7 dsDNA viruses from 7 different families which were involved in more than one endogenization Event except for textit{Phycodnavirida}e. Red point mean that the same function is found within candidates and in the viral dsDNA GO terms repertoire. The size of the dots represents the proportion of the GO terms in the dsDNA virus repertoire. }
%\end{figure}

% \usepackage{array}
% \usepackage{graphicx}
% \usepackage{booktabs}


\begin{table}
\centering
\tiny
\caption[Paper1:RNA EVEs distribution along scaffolds]{\textbf{EVEs distribution according to putative non-segmented single-stranded RNA virus donor and their genomic position}. For each EVE, we retrieved the position of the homologous ORF in the virus genome identified after a blastp search (first hit). The information on the position of the ORFs was retrieved either from \cite{shi_redefining_2016}, from the ICTV reports or manually after recovery of the viral assembly in NCBI, ORF annotation with getorf (ORF length min= 150bp) and blastp to confirm the position of the ORFs and their functions. The position of the ORF in the free-living virus genome is reported in the column "ORF position", when the genome was incomplete, we inferred the position of the ORF with respect to the position of the homolog in the closest complete viral genome. The number of EVEs corresponds to the number of EVEs counting paralogs only once (i.e. counting only one EVE per species per gene cluster).}
\resizebox{\linewidth}{!}{%
\begin{tabular}{>{\hspace{0pt}}m{0.092\linewidth}>{\hspace{0pt}}m{0.179\linewidth}>{\hspace{0pt}}m{0.094\linewidth}>{\hspace{0pt}}m{0.24\linewidth}>{\hspace{0pt}}m{0.159\linewidth}>{\hspace{0pt}}m{0.088\linewidth}>{\hspace{0pt}}m{0.065\linewidth}} 
\toprule
Family & Closest viral blast hit & Protein ID & Protein Name & ORF position & Source & Nb EVEs  \\ 
\hline
\textit{Artoviridae} & Pteromalus puparum peropuvirus & YP\_009505431 & Glycosylated matrix protein M (U3) & 3 out 5 & ICTV & 19 \\
\textit{Artoviridae} & Pteromalus puparum peropuvirus & YP\_009505433 & RNA pol (L) & 5 out 5 & ICTV & 9 \\
\textit{Artoviridae} & Beihai rhabdo like virus 1 & YP\_009333442 & Phosphoprotein (U2) & 2 out 3 & ICTV & 1 \\ 
\hline
\textit{Bornaviridae} & Estrildid finch bornavirus 1 & YP\_009505428 & RNA pol (L) & 4 out
  4 & Manually & 8 \\ 
\hline
\textit{Chuviridae} & Hubei chuvirus like 1 & YP\_009337906 & Nucleoprotein (N) & Incomplete (putative 3 out 3) & Manually & 24 \\
\textit{Chuviridae} & Hubei coleoptera virus  3 & YP\_009336865 & Nucleoprotein (N) & 2 out 3 & Shi et al 2016 & 6 \\
\textit{Chuviridae} & Hubei chuvirus like 3 & YP\_009337091 & Nucleoprotein (N) & 3 out 3 & Shi et al 2016 & 2 \\
\textit{Chuviridae} & Hubei coleoptera virus 3 & YP\_009336866 & RNA pol (L) & 3 out 3 & Shi et al 2016 & 2 \\
\textit{Chuviridae} & Lonestar tick chuvirus & YP\_009254003 & Nucleoprotein (N) & 3 out 3 & Manually & 1 \\
\textit{Chuviridae} & Hubei odonate virus 11 & YP\_009336948 & Nucleoprotein (N) & 3 out 3 & Shi et al 2016 & 1 \\ 
\hline
\textit{Lispiviridae} & Tacheng tick virus 6 & YP\_009304417 & ORF1 & 1 out 5 & Manually & 5 \\
\textit{Lispiviridae} & Hubei rhabdo like virus 3 & YP\_009336884 & Glyceraldehyde-3-phosphate dehydrogenase & 1 out 5 & Manually & 7 \\
\textit{Lispiviridae} & Hubei rhabdo like virus 3 & YP\_009336887 & Glycoprotein (G) & 3 out 5 & Manually & 2 \\
\textit{Lispiviridae} & Hubei rhabdo like virus 3 & YP\_009336889 & RNA pol (L) & 5 out 5 & Manually & 1 \\ 
\hline
\textit{Nyamiviridae} & Midway nyavirus & YP\_002905336 & Nucleoprotein (N) & 1 out 6 & ICTV & 8 \\
\textit{Nyamiviridae} & Orinoco orinovirus & YP\_009666283 & Nucleoprotein (N) & 1 out 4 & Manually & 8 \\
\textit{Nyamiviridae} & Sierra Nevada nyavirus & YP\_009044206 & Nucleoprotein (N) & 1 out 6 & ICTV & 5 \\
\textit{Nyamiviridae} & Nyamanini nyavirus & YP\_002905342 & Nucleoprotein (N) & 1 out 6 & ICTV & 4 \\
\textit{Nyamiviridae} & Midway nyavirus & YP\_002905331 & RNA pol (L) & 6 out 6 & ICTV & 1 \\
\textit{Nyamiviridae} & Wenzhou Crab Virus 1 & YP\_009304556 & Nucleoprotein (N) & 1 out 4 & Manually & 1 \\
\textit{Nyamiviridae} & Beihai rhabdo like virus 3 & YP\_009666292 & RNA pol (L) & 5 out 5 & Shi et al 2016 & 1 \\ 
\hline
\textit{Rhabdoviridae} & Wugan Ant virus & YP\_009304559 & RNA pol (L) & Incomplete (putative 5 out 5) & Manually & 2 \\
\textit{Rhabdoviridae} & Wuhan insect virus 7 & YP\_009301743 & RNA pol (L) & 5 out
  5 & Manually & 1 \\
\textit{Rhabdoviridae} & Sanxia Water Strider Virus 5 & YP\_009289351 & Glycoprotein (G) & 4 out
  5 & Manually & 1 \\
\textit{Rhabdoviridae} & Muscina stabulans sigmavirus & YP\_009664711 & RNA pol (L) & Incomplete (putative 5 out 5)  & Manually & 1 \\
\bottomrule
\end{tabular}
}
\label{tab:single-stranded_RNA_orf_location}
\end{table}


\begin{figure}[!htpbt]
\includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Multiple_new_integration_events_gene_plot.pdf}
\centering
\caption[Paper1:EVE distribution along scaffolds of two ant species]{\textbf{Candidate EVEs in two ant species}. In grey are displayed the eukaryotic genes predicted by Augustus, with a dark color for exons, and light for introns. In black are displayed the transposable elements predicted by sequence homologies with the RepeatPeps protein database. The size of the scaffold is displayed below the name of each scaffold. The letter followed by the scaffold name refers to the scoring given to the scaffold based on coverage and gene/ET presence information (see details in \hyperref[sec:MM-5]{Materials and methods}). For the sake of representation, all scaffolds are represented at the different scale, the exact coordinates of the elements are referred in the abscissa which corresponds to the coordinates in base pairs. Annotation is indicated above the arrows.}
\label{figure:Multiple_new_endogenization_events_gene_plot}
\end{figure}


\begin{figure}[!htpbt]
\includegraphics[width=.95\linewidth,height=.95\textheight,keepaspectratio]{PhD-master/figures/PoEFV_PoFV_table_for_GGGenes_plot.pdf}\centering
\caption[Paper1:Filamentous EVE distribution along \textit{P.orseoliae} scaffolds]{\scriptsize \textbf{Genomic environment for the EVEs detected in \textit{Platygaster orseoliae}}. The plot show regions homologous to viral ORFs in the Platygaster orseoliae filamentous virus (PoFV) genome (A). The colored regions correspond to the predicted ORFs in the PoFV genome (gray ORFS in PoFV scaffolds mean that no homologous EVEs were found in the genome of \textit{P.orseoliae}.) (B), so the closer the colours, the closer the ORFs were initially in the PoFV genome. In grey are displayed the eukaryotic genes predicted by Augustus, with a dark color for exons, and light for introns. In black are displayed the transposable elements predicted by sequence homologies with the RepeatPeps protein database (max E-value =5.866e-18). The letter followed by the scaffold name refers to the scoring given to the scaffold based on coverage and gene/ET presence information (see details in \hyperref[sec:MM-5]{Materials and methods}). The exact coordinates of the elements are referred to the x abscissa, which corresponds to the coordinates in base pairs and can be found on the GitHub file : \href{https://github.com/BenjaminGuinet/PhD_defense/blob/main/Supplementary_paper1/All_PoFV_PoEFV_loci_informations.txt}{All\_PoFV\_PoEFV\_loci\_informations.txt}. Annotation is indicated in or next to the arrows. The number below each EVE correspond to the homologous ORF number in the PoFV genome. Numbers are colored in red if the EVE has at least one paralog and if the computed \textit{dN/dS} is below 1, suggesting purifying selection in the \textit{P.orseoliae} genome. Arrow with black borders correspond to EVEs showing a complete ORF ($>$50\% of the size of the best PoFV ORF). Hatched arrows correspond to pseudogenized EVEs (with premature stop codons).   The colour difference between black and white for the names of the proteins is for visual purposes only.}
\label{figure:Porseoliae_endogenization_events_gene_plot}
\end{figure}


\begin{figure}[!ht]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Heatmap_control.pdf}
\caption[Paper1:Distribution of domesticated EVEs in the literature used as control]{\textbf{Heatmap representing the viral genes known to be domesticated by Hymenoptera}. 
The panel (A) refers to the four known cases (\textit{Venturia canescens}, \textit{Fopius arisanus}, \textit{Cotesia congregata} and \textit{Microplitis demolitor}) involving Nudivirus donors while the panel (B) refers to the known case involving LbFV donors in three \textit{Leptopilina} species. Complete parasitoid wasp genomes information were available for \textit{Microplitis demolitor}, \textit{Venturia canescens}, \textit{Fopius arisanus} and \textit{Cotesia congregata}, while only partial genomic data were available for \textit{Chelonus inanitus}. Each row indicates a gene which has been identified previously as being endogenized in at least one species. In (A), the first four columns indicate whether the gene is a core gene for baculoviruses (Bv), Nudiviruses (Nd), alpha-nudiviruses (alpha-Nv) or beta-nudivirus (beta-nv). The following columns indicate the presence of each gene based on the literature (in blue) and based on our pipeline (columns with a star symbol). The colors indicate the inferred selection pressure acting on each gene (\textit{dN/dS}) and the letters A, B, C, D, E, and X represent the degree of confidence in the endogenization. Capital letters indicate that this gene is present in a scaffold that contains other candidate genes. When the box is framed in black, it means that the gene is expressed (TPM$>$1000).}
\label{figure:Heatmap_control}

\end{figure}

\begin{figure}[!ht]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/dsDNA_violin_plots_corrected_rates.pdf}
\caption[Paper1:Violin plot GLM distribution of corrected dEvent ratios]{\footnotesize\textbf{Violin plots of the posterior distribution of dEvents GLM coefficients in relation to wasp lifestyle (corrected for Events rates) }. The ectoparasitoid lifestyle is in yellow, the endoparasitoid lifestyle is in green, and the free-living lifestyle is in blue (the intercept). Coefficients have been transformed into exponential and correspond to the posterior distribution of the coefficients of a binomial logistic regression GLM model, where the lifestyle free-living stand for the intercept. The Y-axis corresponds to the multiplicative factor of the number of dEvents (corrected for Events rates) correlative to free-living species. The coefficients are derived from 1000 GLM models adjusted on 1000 randomly selected probable scenarios ($>$90 CI) of ancestral states at nodes. Branches from nodes older than 160 million years have been removed from the dataset. The ROPE\% is the percentage of the posterior distribution of coefficients below the intercept. The posterior distribution of the interaction coefficients between lifestyles and branch size were not informative, and the branch size factor was therefore added as an additive effect to the model. \textbf{A}- The corrected coefficient within all dEvents, \textbf{B}- The corrected coefficient within all dEvents without the control genomes, \textbf{C}- The corrected coefficient within all dEvents present in a scaffold annotated with a score A, \textbf{D}- The corrected coefficient within all dEvents present in a scaffold annotated with a score A and without the control genomes.}
\label{figure:dsDNA_violin_plots_corrected_rates}
\end{figure}


\begin{figure}[!ht]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Campopleginae_synteny.pdf}
\caption[Paper1:IVSPER gene distribution along Campopleginae scaffolds]{ \textbf{IVSPER genes identified in the Campopleginae genome}. The figure compares the synteny of the IVSPER between Hyposoter didymator ichnovirus (HdIV) and  the Campoplegninae of our dataset. Homologous genes with
synteny between the two species are indicated by grey shading. The direction of the arrows corresponds to the sense and anti-sense strand. The color of the boxes is unique to each IVSPER. }
\label{figure:Campoplegniae_synteny}
\end{figure}

\begin{figure}[!ht]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Ophioniformes_phylogeny.pdf}
\caption[Paper1:Recapitulation of Ichnovirus within Ophioniformes wasps]{\textbf{Cladogram of the Ophioniformes group, illustrating the two independent endogenization events of two unknown viruses in Banchinae and Campopleginae lineages}. The phylogeny includes 12 subfamilies of the Ophioniformes group within the superfamily \textitit{Ichneumonoidea}. Several species of these subfamilies have been examined for the presence of ichnovirus-like polydnaviruses: by negative staining of calyx fluid (N), TEM of ovarian sections (S), visual examination of the calyx fluid (CF), probes from ichnovirus replication or structural proteins (probes) or by IVSPER sequence homology on whole genome assemblies (WG). The subfamilies and species in blue correspond to those positive for a dsDNA virus endogenization from unknown origin (ichnovirus-like). The others (in black) were negative for endogenized ichnovirus-like elements.
The phylogeny is inspired from \cite{sharanowski_phylogenomics_2021},  and the data reported comes from \cite{sharanowski_phylogenomics_2021,beliveau_genomic_2015,cusson_genomics_2012, legeai_genomic_2020}.}.
\label{figure:Ophioniformes_phylogeny}
\end{figure}

\begin{figure}[!ht]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Bioinformatic_pipeline.pdf}
\caption[Paper1:Simplified summary of the bioinformatics pipeline for the detection of EVEs and dEVEs]{Simplified summary of the bioinformatics pipeline for the detection and validation of candidates for endogenization
and domestication.}
\label{figure:Bioinformatic_pipeline}
\end{figure}

\begin{figure}[!htpb]
 \centering
  \includegraphics[width=\linewidth,height=\textheight,keepaspectratio]{PhD-master/figures/Event_assignation_examples.pdf}
\caption[Paper1:Canonical examples of endogenization events]{\textbf{Canonical examples of endogenization events inferred by our pipeline}. 
The column "Sp names" contains the species name, followed by the name of the scaffold in which the EVE has been identified. The "Viral family" column refers to the putative viral family that donated the EVEs. The column "Cluster number" corresponds to the number of the corresponding cluster phylogeny (thus the EVE phylogeny). The "Monophyletic clade number" column corresponds to the number of the monophyletic clade within a cluster (can be a single locus or  multiple loci). The column "Event number" is the number given to single/multiple EVEs that derive from the same endogenization event.}
\label{figure:Event_assignation_examples}
\end{figure}


%\section{Article}

%\includepdf[pages=-,pagecommand={}]{PhD-master/PDF_files/papier-108.pdf}

